From 62c5246d899f43612da056b79fdf949cadd690e8 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 21 Nov 2020 01:30:06 -0500
Subject: [PATCH 01/78] disable checkout_nacl

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index f8e308ef47374..40c8e81d18e51 100644
--- a/DEPS
+++ b/DEPS
@@ -113,7 +113,7 @@ vars = {
 
   # Check out and download nacl by default, unless on an arm mac.
   # This can be disabled e.g. with custom_vars.
-  'checkout_nacl': 'not (host_os == "mac" and host_cpu == "arm64")',
+  'checkout_nacl': False,
 
   # By default, do not check out src-internal. This can be overridden e.g. with
   # custom_vars.
-- 
2.34.1

