From dcbbc8c072598edecf048f4a1d70f507e0ebee7e Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 21 Nov 2020 01:30:06 -0500
Subject: [PATCH 01/74] disable checkout_nacl

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index af0debed4d07..d87a070c5405 100644
--- a/DEPS
+++ b/DEPS
@@ -93,7 +93,7 @@ vars = {
 
   # Check out and download nacl by default. This can be disabled e.g. with
   # custom_vars.
-  'checkout_nacl': True,
+  'checkout_nacl': False,
 
   # By default, do not check out src-internal. This can be overridden e.g. with
   # custom_vars.
-- 
2.30.2

