From aa68c858ddd93b7b16f1ba416c2ef4831db19a94 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 21 Nov 2020 01:30:06 -0500
Subject: [PATCH 01/79] disable checkout_nacl

---
 DEPS | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/DEPS b/DEPS
index d01f037f0e6c5..18795134eb004 100644
--- a/DEPS
+++ b/DEPS
@@ -98,9 +98,7 @@ vars = {
   # be overridden by gclient variables.
   'checkout_google_benchmark': False,
 
-  # Check out and download nacl by default, unless on an arm mac.
-  # This can be disabled e.g. with custom_vars.
-  'checkout_nacl': 'not (host_os == "mac" and host_cpu == "arm64")',
+  'checkout_nacl': False,
 
   # By default, do not check out src-internal. This can be overridden e.g. with
   # custom_vars.
-- 
2.32.0

