From 0b73ddcab4eced6c66b2c8cfde5be4337a409cdb Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 22 Dec 2016 07:15:34 -0500
Subject: [PATCH 03/58] enable -fwrapv in Clang for non-UBSan builds

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index a56b237b0a41..c5a13571f3c2 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -288,6 +288,10 @@ config("compiler") {
       }
     }
 
+    if (is_clang && !is_ubsan && !is_ubsan_security) {
+      cflags += [ "-fwrapv" ]
+    }
+
     # Linker warnings.
     if (fatal_linker_warnings && !is_mac && !is_ios && current_os != "aix") {
       ldflags += [ "-Wl,--fatal-warnings" ]
-- 
2.28.0

