From 94896c8fe1d07874e0d8767058376373a5bc6e8b Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 8 Apr 2020 20:48:17 -0400
Subject: [PATCH 04/63] enable -ftrivial-auto-var-init=zero

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 4c89626cd40b..1931d0a7a460 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -294,6 +294,10 @@ config("compiler") {
       cflags += [ "-fwrapv" ]
     }
 
+    if (is_clang) {
+      cflags += [ "-ftrivial-auto-var-init=zero", "-enable-trivial-auto-var-init-zero-knowing-it-will-be-removed-from-clang" ]
+    }
+
     # Linker warnings.
     if (fatal_linker_warnings && !is_mac && !is_ios && current_os != "aix") {
       ldflags += [ "-Wl,--fatal-warnings" ]
-- 
2.29.2

