From 8821ddbbe73a7fd0026a805dbcc3302a269df36d Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 22 Dec 2016 07:15:34 -0500
Subject: [PATCH 04/74] enable -fwrapv in Clang for non-UBSan builds

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 21b3599e0356..9f445ba584a2 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -311,6 +311,10 @@ config("compiler") {
       }
     }
 
+    if (is_clang && !is_ubsan && !is_ubsan_security) {
+      cflags += [ "-fwrapv" ]
+    }
+
     # Linker warnings.
     if (fatal_linker_warnings && !is_apple && current_os != "aix") {
       ldflags += [ "-Wl,--fatal-warnings" ]
-- 
2.30.1

