From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Wed, 31 Jul 2024 12:35:24 +0000
Subject: [PATCH] enable stack clash protection for arm64 targets

---
 build/config/compiler/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 01b278c50f688..3321a77105e16 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -469,6 +469,10 @@ config("compiler") {
       }
     }
 
+    if (target_cpu == "arm64") {
+      cflags += [ "-fstack-clash-protection" ]
+    }
+
     if (use_lld) {
       ldflags += [ "-fuse-ld=lld" ]
       if (lld_path != "") {
