From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 21 May 2020 14:07:54 -0400
Subject: [PATCH] disable broken warning for auto var init

---
 build/config/compiler/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 279784030e8ac..177081f2a8762 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -396,7 +396,7 @@ config("compiler") {
     }
 
     if (is_clang) {
-      cflags += [ "-ftrivial-auto-var-init=zero" ]
+      cflags += [ "-ftrivial-auto-var-init=zero", "-Wno-unused-command-line-argument" ]
     }
 
     # Linker warnings.
