From 78cfcd8be84637fdd0966a45db72df87eb871cc1 Mon Sep 17 00:00:00 2001
From: Zoraver Kang <zkang@wpi.edu>
Date: Fri, 16 Aug 2019 02:03:45 -0400
Subject: [PATCH] redirect settings help icon

---
 .../chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
index 958c80d9fb6ec..f22b01366a6eb 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
@@ -33,7 +33,7 @@ import javax.annotation.Nonnull;
  */
 public class HelpAndFeedbackLauncherImpl implements HelpAndFeedbackLauncher {
     protected static final String FALLBACK_SUPPORT_URL =
-            "https://support.google.com/chrome/topic/6069782";
+            "https://grapheneos.org/usage#web-browsing";
     private static final String TAG = "HelpAndFeedback";
 
     private static HelpAndFeedbackLauncher sInstance;
