From 6dfb6df338efa3bd4e808987713c42bc822098b3 Mon Sep 17 00:00:00 2001
From: Zoraver Kang <zkang@wpi.edu>
Date: Fri, 16 Aug 2019 02:03:45 -0400
Subject: [PATCH] redirect settings help icon

---
 .../chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
index ccf04eb1168ed..5734f98f8bdde 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
@@ -31,7 +31,7 @@ import javax.annotation.Nonnull;
  */
 public class HelpAndFeedbackLauncherImpl implements HelpAndFeedbackLauncher {
     protected static final String FALLBACK_SUPPORT_URL =
-            "https://support.google.com/chrome/topic/6069782";
+            "https://grapheneos.org/usage#web-browsing";
     private static final String TAG = "HelpAndFeedback";
 
     private static HelpAndFeedbackLauncher sInstance;
