From 7fa95462a938d07911d63b807618d8006c0d672f Mon Sep 17 00:00:00 2001
From: "a@b.c" <a@b.c>
Date: Sat, 16 Jan 2021 15:38:04 +0000
Subject: [PATCH 11/75] redirect settings help icon

---
 .../chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
index ccf04eb1168e..5734f98f8bdd 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/feedback/HelpAndFeedbackLauncherImpl.java
@@ -31,7 +31,7 @@ import javax.annotation.Nonnull;
  */
 public class HelpAndFeedbackLauncherImpl implements HelpAndFeedbackLauncher {
     protected static final String FALLBACK_SUPPORT_URL =
-            "https://support.google.com/chrome/topic/6069782";
+            "https://grapheneos.org/usage#web-browsing";
     private static final String TAG = "HelpAndFeedback";
 
     private static HelpAndFeedbackLauncher sInstance;
-- 
2.31.1

