From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Wed, 24 Jan 2024 11:22:54 +0000
Subject: [PATCH] Checkout PGO profiles

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index 2ef294f2050ad..aeb7785f9eaa7 100644
--- a/DEPS
+++ b/DEPS
@@ -149,7 +149,7 @@ vars = {
   'checkout_clang_coverage_tools': False,
 
   # Fetch the pgo profiles to optimize official builds.
-  'checkout_pgo_profiles': False,
+  'checkout_pgo_profiles': True,
 
   # Fetch clang-tidy into the same bin/ directory as our clang binary.
   'checkout_clang_tidy': False,
