From c967fdb1ed105dd39d4601bd89191243c4acac40 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 15 Apr 2021 02:14:37 -0400
Subject: [PATCH 13/79] remove Help & feedback menu entry

---
 .../browser/app/appmenu/AppMenuPropertiesDelegateImpl.java      | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
index bac7ac6c9b61c..c4124557a4772 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java
@@ -352,6 +352,8 @@ public class AppMenuPropertiesDelegateImpl implements AppMenuPropertiesDelegate
         // Only display reader mode settings menu option if the current page is in reader mode.
         menu.findItem(R.id.reader_mode_prefs_id).setVisible(shouldShowReaderModePrefs(currentTab));
 
+        menu.findItem(R.id.help_id).setVisible(false);
+
         // Only display the Enter VR button if VR Shell Dev environment is enabled.
         menu.findItem(R.id.enter_vr_id).setVisible(shouldShowEnterVr());
 
-- 
2.32.0

