From 6091f9b0e8734d38ea1aec2a95afd74c95665fd7 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 24 Nov 2016 08:19:03 -0500
Subject: [PATCH 14/73] disable first run welcome page

---
 .../chrome/browser/firstrun/FirstRunActivity.java         | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/firstrun/FirstRunActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/firstrun/FirstRunActivity.java
index 54469ad0153b..01c44da6b591 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/firstrun/FirstRunActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/firstrun/FirstRunActivity.java
@@ -128,10 +128,10 @@ public class FirstRunActivity extends FirstRunActivityBase implements FirstRunPa
      * Defines a sequence of pages to be shown (depending on parameters etc).
      */
     private void createPageSequence() {
-        mPages.add(shouldCreateEnterpriseCctTosPage()
-                        ? new TosAndUmaFirstRunFragmentWithEnterpriseSupport.Page()
-                        : new ToSAndUMAFirstRunFragment.Page());
-        mFreProgressStates.add(FRE_PROGRESS_WELCOME_SHOWN);
+        //mPages.add(shouldCreateEnterpriseCctTosPage()
+                        //? new TosAndUmaFirstRunFragmentWithEnterpriseSupport.Page()
+                        //: new ToSAndUMAFirstRunFragment.Page());
+        //mFreProgressStates.add(FRE_PROGRESS_WELCOME_SHOWN);
         // Other pages will be created by createPostNativeAndPoliciesPageSequence() after
         // native and policy service have been initialized.
     }
-- 
2.30.1

