From 0a70c6c84a098d0a65df590f9bda22213af7d782 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 6 Mar 2018 00:27:41 -0500
Subject: [PATCH] disable showing popular sites by default

---
 components/ntp_tiles/features.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/ntp_tiles/features.cc b/components/ntp_tiles/features.cc
index 8b29a14102ba0..48df4146cd5d0 100644
--- a/components/ntp_tiles/features.cc
+++ b/components/ntp_tiles/features.cc
@@ -18,7 +18,7 @@ const char kIOSPopularSitesExcludePopularAppsParam[] =
 
 BASE_FEATURE(kPopularSitesBakedInContentFeature,
              "NTPPopularSitesBakedInContent",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNtpMostLikelyFaviconsFromServerFeature,
              "NTPMostLikelyFaviconsFromServer",
@@ -26,7 +26,7 @@ BASE_FEATURE(kNtpMostLikelyFaviconsFromServerFeature,
 
 BASE_FEATURE(kUsePopularSitesSuggestions,
              "UsePopularSitesSuggestions",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kIOSPopularSitesImprovedSuggestions,
              "IOSPopularSitesImprovedSuggestions",
