From 26e8a67018b645b7ea3d1721d0aaf20f4de25f97 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 4 Jul 2019 18:11:27 -0400
Subject: [PATCH 26/58] disable media router media remoting by default

---
 chrome/browser/profiles/profile.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index d49a8c201d7a..cd55ab8b6f1f 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -330,7 +330,7 @@ void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
       prefs::kMediaRouterFirstRunFlowAcknowledged,
       false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
-  registry->RegisterBooleanPref(prefs::kMediaRouterMediaRemotingEnabled, true);
+  registry->RegisterBooleanPref(prefs::kMediaRouterMediaRemotingEnabled, false);
   registry->RegisterListPref(prefs::kMediaRouterTabMirroringSources);
 
   registry->RegisterDictionaryPref(prefs::kWebShareVisitedTargets);
-- 
2.27.0

