From 69f7b79a1f7c244600c91186328be979d4d43f94 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 4 Jul 2019 18:11:27 -0400
Subject: [PATCH 31/71] disable media router media remoting by default

---
 chrome/browser/profiles/profile.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 759225ea6052..2613f2e35685 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -345,7 +345,7 @@ void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
       media_router::prefs::kMediaRouterEnableCloudServices, false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   registry->RegisterBooleanPref(
-      media_router::prefs::kMediaRouterMediaRemotingEnabled, true);
+      media_router::prefs::kMediaRouterMediaRemotingEnabled, false);
   registry->RegisterListPref(
       media_router::prefs::kMediaRouterTabMirroringSources);
 #endif
-- 
2.29.2

