From babb628694911d5169265d67a9664ed871de245e Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 17 Jun 2019 13:14:22 -0400
Subject: [PATCH 40/57] always use local new tab page

---
 chrome/browser/search/search.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/search/search.cc b/chrome/browser/search/search.cc
index 20b0f8a4c152..7d150b0037bd 100644
--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -160,7 +160,7 @@ bool ShouldShowLocalNewTab(Profile* profile) {
 #if !defined(OS_ANDROID)
   return DefaultSearchProviderIsGoogle(profile);
 #else
-  return false;
+  return true;
 #endif
 }
 
-- 
2.27.0

