From 519c17f73657911ca047b06b5bbcdd26e63888d1 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 23 Dec 2020 06:00:50 -0500
Subject: [PATCH] enable split cache by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index ae840209abe09..b0cca595e567b 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -102,7 +102,7 @@ const base::Feature kSplitCacheByIncludeCredentials{
     "SplitCacheByIncludeCredentials", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kSplitCacheByNetworkIsolationKey{
-    "SplitCacheByNetworkIsolationKey", base::FEATURE_DISABLED_BY_DEFAULT};
+    "SplitCacheByNetworkIsolationKey", base::FEATURE_ENABLED_BY_DEFAULT};
 
 const base::Feature kSplitHostCacheByNetworkIsolationKey{
     "SplitHostCacheByNetworkIsolationKey", base::FEATURE_DISABLED_BY_DEFAULT};
