From 1d0e397aab7af2bee87df08b28c335589cdaf5e2 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 23 Dec 2020 06:00:50 -0500
Subject: [PATCH] enable split cache by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index a43cca47a5715..c15860b5bf432 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -118,7 +118,7 @@ const base::Feature kSplitCacheByIncludeCredentials{
     "SplitCacheByIncludeCredentials", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kSplitCacheByNetworkIsolationKey{
-    "SplitCacheByNetworkIsolationKey", base::FEATURE_DISABLED_BY_DEFAULT};
+    "SplitCacheByNetworkIsolationKey", base::FEATURE_ENABLED_BY_DEFAULT};
 
 const base::Feature kSplitHostCacheByNetworkIsolationKey{
     "SplitHostCacheByNetworkIsolationKey", base::FEATURE_DISABLED_BY_DEFAULT};
