From 16bc87764a7c18d141f04b923975703a3ad5dd16 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 23 Dec 2020 06:00:50 -0500
Subject: [PATCH 43/75] enable split cache by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 4cb6131f39d8..0fda76e36fd6 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -71,7 +71,7 @@ const base::Feature kNetworkQualityEstimator{"NetworkQualityEstimator",
                                              base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kSplitCacheByNetworkIsolationKey{
-    "SplitCacheByNetworkIsolationKey", base::FEATURE_DISABLED_BY_DEFAULT};
+    "SplitCacheByNetworkIsolationKey", base::FEATURE_ENABLED_BY_DEFAULT};
 
 const base::Feature kSplitHostCacheByNetworkIsolationKey{
     "SplitHostCacheByNetworkIsolationKey", base::FEATURE_DISABLED_BY_DEFAULT};
-- 
2.31.1

