From b9372a535cfc7d5507b4a91941537850c53b515a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 17:48:49 -0400
Subject: [PATCH 43/58] show download prompt again by default

---
 .../chrome/browser/download/DownloadLocationCustomView.java   | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java
index f9605b8e95f2..a7c850e5d903 100644
--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java
+++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java
@@ -54,10 +54,6 @@ public class DownloadLocationCustomView
     void initialize(@DownloadLocationDialogType int dialogType, File suggestedPath) {
         mDialogType = dialogType;
 
-        // Automatically check "don't show again" the first time the user is seeing the dialog.
-        boolean isInitial = DownloadLocationDialogBridge.getPromptForDownloadAndroid()
-                == DownloadPromptStatus.SHOW_INITIAL;
-        mDontShowAgain.setChecked(isInitial);
         mDontShowAgain.setOnCheckedChangeListener(this);
 
         mFileName.setText(suggestedPath.getName());
-- 
2.27.0

