From 08f836d2255933a519f838a03a5a9480070fa53a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:46:41 -0400
Subject: [PATCH 45/56] remove navigation error preference

---
 .../settings/sync/SyncAndServicesSettings.java    | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java
index 4c601d9c2647..43bf53fdb18f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java
@@ -198,16 +198,19 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         mSearchSuggestions.setOnPreferenceChangeListener(this);
         mSearchSuggestions.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
 
+        PreferenceCategory servicesCategory =
+                (PreferenceCategory) findPreference(PREF_SERVICES_CATEGORY);
+
         mNavigationError = (ChromeSwitchPreference) findPreference(PREF_NAVIGATION_ERROR);
-        mNavigationError.setOnPreferenceChangeListener(this);
-        mNavigationError.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mNavigationError.setOnPreferenceChangeListener(this);
+        //mNavigationError.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mNavigationError);
+        mNavigationError = null;
 
         mSafeBrowsing = (ChromeSwitchPreference) findPreference(PREF_SAFE_BROWSING);
         mSafeBrowsing.setOnPreferenceChangeListener(this);
         mSafeBrowsing.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
 
-        PreferenceCategory servicesCategory =
-                (PreferenceCategory) findPreference(PREF_SERVICES_CATEGORY);
         mPasswordLeakDetection =
                 (ChromeSwitchPreference) findPreference(PREF_PASSWORD_LEAK_DETECTION);
         mPasswordLeakDetection.setOnPreferenceChangeListener(this);
@@ -537,8 +540,8 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         //updateSyncPreferences();
 
         mSearchSuggestions.setChecked(mPrefServiceBridge.getBoolean(Pref.SEARCH_SUGGEST_ENABLED));
-        mNavigationError.setChecked(
-                mPrefServiceBridge.getBoolean(Pref.ALTERNATE_ERROR_PAGES_ENABLED));
+        //mNavigationError.setChecked(
+                //mPrefServiceBridge.getBoolean(Pref.ALTERNATE_ERROR_PAGES_ENABLED));
         mSafeBrowsing.setChecked(mPrefServiceBridge.getBoolean(Pref.SAFE_BROWSING_ENABLED));
 
         updateLeakDetectionAndSafeBrowsingReportingPreferences();
-- 
2.26.0

