From 1c7788c6c73359b5c48f01f6b54bb97aff1e2711 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 27 Nov 2020 02:43:37 -0500
Subject: [PATCH 46/74] disable autofill assistant by default

---
 .../autofill_assistant/AutofillAssistantPreferencesUtil.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/features/autofill_assistant/public/java/src/org/chromium/chrome/browser/autofill_assistant/AutofillAssistantPreferencesUtil.java b/chrome/android/features/autofill_assistant/public/java/src/org/chromium/chrome/browser/autofill_assistant/AutofillAssistantPreferencesUtil.java
index 59d1ea6a35ab..d1ac8a844d50 100644
--- a/chrome/android/features/autofill_assistant/public/java/src/org/chromium/chrome/browser/autofill_assistant/AutofillAssistantPreferencesUtil.java
+++ b/chrome/android/features/autofill_assistant/public/java/src/org/chromium/chrome/browser/autofill_assistant/AutofillAssistantPreferencesUtil.java
@@ -23,7 +23,7 @@ public class AutofillAssistantPreferencesUtil {
     /** Checks whether the Autofill Assistant switch preference in settings is on. */
     static boolean isAutofillAssistantSwitchOn() {
         return SharedPreferencesManager.getInstance().readBoolean(
-                ChromePreferenceKeys.AUTOFILL_ASSISTANT_ENABLED, true);
+                ChromePreferenceKeys.AUTOFILL_ASSISTANT_ENABLED, false);
     }
 
     /** Checks whether proactive help is enabled. */
-- 
2.31.1

