From 6335287624f03681eb7c5a7c96a156d9aec56c20 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 17:48:49 -0400
Subject: [PATCH 47/63] show download prompt again by default

---
 .../browser/download/dialogs/DownloadLocationCustomView.java  | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
index f633d3739285..07853777d5fb 100644
--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
+++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
@@ -63,10 +63,6 @@ public class DownloadLocationCustomView
             CharSequence title) {
         mDialogType = dialogType;
 
-        // Automatically check "don't show again" the first time the user is seeing the dialog.
-        boolean isInitial = DownloadDialogBridge.getPromptForDownloadAndroid()
-                == DownloadPromptStatus.SHOW_INITIAL;
-        mDontShowAgain.setChecked(isInitial);
         mDontShowAgain.setOnCheckedChangeListener(this);
 
         mFileName.setText(suggestedPath.getName());
-- 
2.29.2

