From 6b4c79ed8e2574c6edb7b1372914c23883ec8397 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:50:03 -0400
Subject: [PATCH 49/55] remove url keyed anonymized data preference

---
 .../browser/settings/sync/SyncAndServicesSettings.java | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java
index 906828b2bae2..90cc9dc07416 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/sync/SyncAndServicesSettings.java
@@ -234,8 +234,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
-        mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
-        mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
+        //mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUrlKeyedAnonymizedData);
+        mUrlKeyedAnonymizedData = null;
 
         mAutofillAssistant = (ChromeSwitchPreference) findPreference(PREF_AUTOFILL_ASSISTANT);
         if (shouldShowAutofillAssistantPreference()) {
@@ -554,8 +556,8 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         //mUsageAndCrashReporting.setChecked(
                 //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
-        mUrlKeyedAnonymizedData.setChecked(
-                UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled());
+        //mUrlKeyedAnonymizedData.setChecked(
+                //UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled());
 
         if (mAutofillAssistant != null) {
             mAutofillAssistant.setChecked(isAutofillAssistantSwitchOn());
-- 
2.26.0

