From 11887d3ac0a82eea75fb63c3a67ebeaba65d683a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 8 Apr 2020 09:04:51 -0400
Subject: [PATCH 49/58] remove password leak detection preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 16537cd885e7..8e300a24c4e6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -216,8 +216,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mPasswordLeakDetection =
                 (ChromeSwitchPreference) findPreference(PREF_PASSWORD_LEAK_DETECTION);
-        mPasswordLeakDetection.setOnPreferenceChangeListener(this);
-        mPasswordLeakDetection.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mPasswordLeakDetection.setOnPreferenceChangeListener(this);
+        //mPasswordLeakDetection.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mPasswordLeakDetection);
+        mPasswordLeakDetection = null;
 
         mSafeBrowsingReporting =
                 (ChromeSwitchPreference) findPreference(PREF_SAFE_BROWSING_SCOUT_REPORTING);
@@ -631,6 +633,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         mSafeBrowsingReporting.setChecked(safe_browsing_enabled
                 && SafeBrowsingBridge.isSafeBrowsingExtendedReportingEnabled());
 
+        if (mPasswordLeakDetection == null) {
+            return;
+        }
+
         boolean has_token_for_leak_check = PasswordUIView.hasAccountForLeakCheckRequest();
         boolean leak_detection_enabled =
                 mPrefServiceBridge.getBoolean(Pref.PASSWORD_LEAK_DETECTION_ENABLED);
-- 
2.28.0

