From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 17 Jun 2019 13:14:22 -0400
Subject: [PATCH] always use local new tab page

---
 chrome/browser/search/search.cc | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/chrome/browser/search/search.cc b/chrome/browser/search/search.cc
index 86564e6286685..bffa07a433ff0 100644
--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -163,6 +163,10 @@ struct NewTabURLDetails {
   NewTabURLDetails(const GURL& url, NewTabURLState state)
       : url(url), state(state) {}
 
+  static bool ShouldUseLocalNewTab() {
+    return true;
+  }
+
   static NewTabURLDetails ForProfile(Profile* profile) {
     // Incognito and Guest profiles have their own New Tab.
     // This function may also be called by other off-the-record profiles that
@@ -174,6 +178,9 @@ struct NewTabURLDetails {
 
 #if BUILDFLAG(IS_ANDROID)
     const GURL local_url;
+    if (ShouldUseLocalNewTab()) {
+      return NewTabURLDetails(local_url, NEW_TAB_URL_VALID);
+    }
 #else
     const bool default_is_google = DefaultSearchProviderIsGoogle(profile);
     const GURL local_url(default_is_google
