From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Tue, 18 Jun 2024 18:50:35 +0000
Subject: [PATCH] Disable top toolbar button Translate option by default

---
 .../settings/RadioButtonGroupAdaptiveToolbarPreference.java  | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/adaptive/settings/RadioButtonGroupAdaptiveToolbarPreference.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/adaptive/settings/RadioButtonGroupAdaptiveToolbarPreference.java
index b10df6c9a86fa..83ee5d507016d 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/adaptive/settings/RadioButtonGroupAdaptiveToolbarPreference.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/adaptive/settings/RadioButtonGroupAdaptiveToolbarPreference.java
@@ -152,8 +152,13 @@ public class RadioButtonGroupAdaptiveToolbarPreference extends Preference
         updateReadAloudButtonVisibility();
         updatePageSummaryButtonVisibility();
         updateOpenInBrowserButtonVisibility();
+        updateRemoveUnneededButtons();
         mButtonsInitialized = true;
     }
+ 
+    private void updateRemoveUnneededButtons() {
+        updateButtonVisibility(mTranslateButton, false);
+    }
 
     @Override
     public void onCheckedChanged(RadioGroup group, int checkedId) {
