From ea33efdf729b040c3a318b73068a430dd73d8205 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:50:03 -0400
Subject: [PATCH 52/58] remove url keyed anonymized data preference

---
 .../sync/settings/SyncAndServicesSettings.java       | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index a42fdf632046..61d9db935464 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -237,8 +237,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
-        mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
-        mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
+        //mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUrlKeyedAnonymizedData);
+        mUrlKeyedAnonymizedData = null;
 
         mAutofillAssistant = (ChromeSwitchPreference) findPreference(PREF_AUTOFILL_ASSISTANT);
         if (shouldShowAutofillAssistantPreference()) {
@@ -559,9 +561,9 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         //mUsageAndCrashReporting.setChecked(
                 //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
-        mUrlKeyedAnonymizedData.setChecked(
-                UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
-                        Profile.getLastUsedRegularProfile()));
+        //mUrlKeyedAnonymizedData.setChecked(
+                //UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
+                        //Profile.getLastUsedRegularProfile()));
 
         if (mAutofillAssistant != null) {
             mAutofillAssistant.setChecked(isAutofillAssistantSwitchOn());
-- 
2.27.0

