From feb81060640a7cf14b364af4d4a05d85072c342b Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 16:56:37 -0400
Subject: [PATCH] stop ignoring download location prompt setting

---
 .../DownloadLocationDialogCoordinator.java        | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationDialogCoordinator.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationDialogCoordinator.java
index c8d406b43e5c2..95d2545cfbd4e 100644
--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationDialogCoordinator.java
+++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationDialogCoordinator.java
@@ -132,21 +132,6 @@ public class DownloadLocationDialogCoordinator implements ModalDialogProperties.
      * @param dirs An list of available download directories.
      */
     private void onDirectoryOptionsRetrieved(ArrayList<DirectoryOption> dirs) {
-        // If there is only one directory available, don't show the default dialog, and set the
-        // download directory to default. Dialog will still show for other types of dialogs, like
-        // name conflict or disk error or if Incognito download warning is needed.
-        if (dirs.size() == 1 && !mLocationDialogManaged
-                && mDialogType == DownloadLocationDialogType.DEFAULT
-                && !shouldShowIncognitoWarning()) {
-            final DirectoryOption dir = dirs.get(0);
-            if (dir.type == DirectoryOption.DownloadLocationDirectoryType.DEFAULT) {
-                assert (!TextUtils.isEmpty(dir.location));
-                DownloadDialogBridge.setDownloadAndSaveFileDefaultDirectory(dir.location);
-                mController.onDownloadLocationDialogComplete(mSuggestedPath);
-            }
-            return;
-        }
-
         // Already showing the dialog.
         if (mDialogModel != null) return;
 
