From ac94fda0f4e97f8314afbafd2c218090cc062e77 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 17 Jun 2019 13:14:22 -0400
Subject: [PATCH 54/79] always use local new tab page

---
 chrome/browser/search/search.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/search/search.cc b/chrome/browser/search/search.cc
index a624563aedd48..a2d3d9a742478 100644
--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -156,7 +156,7 @@ bool ShouldShowLocalNewTab(Profile* profile) {
 #if !defined(OS_ANDROID)
   return DefaultSearchProviderIsGoogle(profile);
 #else
-  return false;
+  return true;
 #endif
 }
 
-- 
2.32.0

