From 512f7a323101c5db9d58470f981fc70cee81e7b6 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 8 Apr 2020 09:04:51 -0400
Subject: [PATCH 55/70] remove password leak detection preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 875bb2a5fc92..f6f208684893 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -233,8 +233,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
             mPasswordLeakDetection =
                     (ChromeSwitchPreference) findPreference(PREF_PASSWORD_LEAK_DETECTION);
-            mPasswordLeakDetection.setOnPreferenceChangeListener(this);
-            mPasswordLeakDetection.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+            //mPasswordLeakDetection.setOnPreferenceChangeListener(this);
+            //mPasswordLeakDetection.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+            removePreference(servicesCategory, mPasswordLeakDetection);
+            mPasswordLeakDetection = null;
 
             mSafeBrowsingReporting =
                     (ChromeSwitchPreference) findPreference(PREF_SAFE_BROWSING_SCOUT_REPORTING);
@@ -652,6 +654,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         mSafeBrowsingReporting.setChecked(safe_browsing_enabled
                 && SafeBrowsingBridge.isSafeBrowsingExtendedReportingEnabled());
 
+        if (mPasswordLeakDetection == null) {
+            return;
+        }
+
         boolean has_token_for_leak_check = PasswordUIView.hasAccountForLeakCheckRequest();
         boolean leak_detection_enabled =
                 mPrefService.getBoolean(Pref.PASSWORD_LEAK_DETECTION_ENABLED);
-- 
2.29.2

