From f7900dbbbbacb4504b4fb4f35d85144fe9054a0f Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 17 Jun 2019 13:14:22 -0400
Subject: [PATCH 55/78] always use local new tab page

---
 chrome/browser/search/search.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/search/search.cc b/chrome/browser/search/search.cc
index a624563aedd4..a2d3d9a74247 100644
--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -156,7 +156,7 @@ bool ShouldShowLocalNewTab(Profile* profile) {
 #if !defined(OS_ANDROID)
   return DefaultSearchProviderIsGoogle(profile);
 #else
-  return false;
+  return true;
 #endif
 }
 
-- 
2.31.1

