From 0d9d3ca82b7a2bdbac67e53051c0225f76ff20a7 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 21 May 2020 14:07:54 -0400
Subject: [PATCH 57/57] disable broken warning for auto var init

---
 build/config/compiler/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 80f28e70651c..97d3d77a9859 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -307,7 +307,7 @@ config("compiler") {
     }
 
     if (is_clang) {
-      cflags += [ "-ftrivial-auto-var-init=zero", "-enable-trivial-auto-var-init-zero-knowing-it-will-be-removed-from-clang" ]
+      cflags += [ "-ftrivial-auto-var-init=zero", "-enable-trivial-auto-var-init-zero-knowing-it-will-be-removed-from-clang", "-Wno-unused-command-line-argument" ]
     }
 
     # Linker warnings.
-- 
2.27.0

