From ba6ff6d82603081d7e9d8e64d94b2100861734d0 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 17:48:49 -0400
Subject: [PATCH 58/78] show download prompt again by default

---
 .../browser/download/dialogs/DownloadLocationCustomView.java  | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
index 4a968c631a22..0dd68bdd5892 100644
--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
+++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
@@ -75,10 +75,6 @@ public class DownloadLocationCustomView
             CharSequence title) {
         mDialogType = dialogType;
 
-        // Automatically check "don't show again" the first time the user is seeing the dialog.
-        boolean isInitial = DownloadDialogBridge.getPromptForDownloadAndroid()
-                == DownloadPromptStatus.SHOW_INITIAL;
-        mDontShowAgain.setChecked(isInitial);
         mDontShowAgain.setOnCheckedChangeListener(this);
 
         mFileName.setText(suggestedPath.getName());
-- 
2.31.1

