From b61fa2aa33294ce1d6766d7abf9a8bf04cccc7fe Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:49:13 -0400
Subject: [PATCH 58/72] remove usage and crash reports preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 908da28f9483..4ce81251a855 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -254,8 +254,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUsageAndCrashReporting =
                 (ChromeSwitchPreference) findPreference(PREF_USAGE_AND_CRASH_REPORTING);
-        mUsageAndCrashReporting.setOnPreferenceChangeListener(this);
-        mUsageAndCrashReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUsageAndCrashReporting.setOnPreferenceChangeListener(this);
+        //mUsageAndCrashReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUsageAndCrashReporting);
+        mUsageAndCrashReporting = null;
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
@@ -596,8 +598,8 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
             updateLeakDetectionAndSafeBrowsingReportingPreferences();
         }
 
-        mUsageAndCrashReporting.setChecked(
-                mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
+        //mUsageAndCrashReporting.setChecked(
+                //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
         mUrlKeyedAnonymizedData.setChecked(
                 UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
                         getProfile()));
-- 
2.30.1

