From 7071f78f9725b312d8e4b40d05087442a252f439 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:50:03 -0400
Subject: [PATCH 60/73] remove url keyed anonymized data preference

---
 .../sync/settings/SyncAndServicesSettings.java       | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 4ce81251a855..4db5b207c824 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -261,8 +261,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
-        mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
-        mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
+        //mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUrlKeyedAnonymizedData);
+        mUrlKeyedAnonymizedData = null;
 
         mAutofillAssistant = (ChromeSwitchPreference) findPreference(PREF_AUTOFILL_ASSISTANT);
         Preference autofillAssistantSubsection = findPreference(PREF_AUTOFILL_ASSISTANT_SUBSECTION);
@@ -600,9 +602,9 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         //mUsageAndCrashReporting.setChecked(
                 //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
-        mUrlKeyedAnonymizedData.setChecked(
-                UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
-                        getProfile()));
+        //mUrlKeyedAnonymizedData.setChecked(
+                //UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
+                        //getProfile()));
 
         if (mAutofillAssistant != null) {
             mAutofillAssistant.setChecked(isAutofillAssistantSwitchOn());
-- 
2.30.1

