From 0038241af028069ec7c10a2d8eb9410af4777dcc Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:49:13 -0400
Subject: [PATCH 62/82] remove usage and crash reports preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index dcdf54309bfa2..cbd82f9a9bef1 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -216,8 +216,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUsageAndCrashReporting =
                 (ChromeSwitchPreference) findPreference(PREF_USAGE_AND_CRASH_REPORTING);
-        mUsageAndCrashReporting.setOnPreferenceChangeListener(this);
-        mUsageAndCrashReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUsageAndCrashReporting.setOnPreferenceChangeListener(this);
+        //mUsageAndCrashReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUsageAndCrashReporting);
+        mUsageAndCrashReporting = null;
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
@@ -539,8 +541,8 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
                     mPrefService.getBoolean(Pref.ALTERNATE_ERROR_PAGES_ENABLED));
         }
 
-        mUsageAndCrashReporting.setChecked(
-                mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
+        //mUsageAndCrashReporting.setChecked(
+                //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
         mUrlKeyedAnonymizedData.setChecked(
                 UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
                         getProfile()));
-- 
2.33.0

