From b44e7665ab16b7d6a0773ef3a1cc6f739d1c7005 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 21:11:17 -0400
Subject: [PATCH] remove translate offer preference

---
 .../browser/language/settings/LanguageSettings.java    | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java b/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
index 3d35cb516928f..dd5509ef85f51 100644
--- a/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
+++ b/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
@@ -79,6 +79,16 @@ public class LanguageSettings extends PreferenceFragmentCompat
             createBasicPreferences(savedInstanceState, rootKey);
         }
 
+        ChromeSwitchPreference translateSwitch =
+                (ChromeSwitchPreference) findPreference(TRANSLATE_SWITCH_KEY);
+        if (translateSwitch != null) {
+            getPreferenceScreen().removePreference(translateSwitch);
+        }
+        String TRANSLATE_SECTION = "translation_settings_section";
+        Preference translateSection = findPreference(TRANSLATE_SECTION);
+        if (translateSection != null) {
+            getPreferenceScreen().removePreference(translateSection);
+        }
         LanguagesManager.recordImpression(LanguagesManager.LanguageSettingsPageType.PAGE_MAIN);
     }
 
