From fffc1a60a7bf02ef0a8237a4116df52b50ce88e4 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:46:41 -0400
Subject: [PATCH 62/83] remove navigation error preference

---
 chrome/common/chrome_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 1e8a97cb1324b..bbce13ff65e91 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -600,7 +600,7 @@ const base::Feature kKernelnextVMs{"KernelnextVMs",
 #if defined(OS_ANDROID)
 COMPONENT_EXPORT(CHROME_FEATURES)
 const base::Feature kLinkDoctorDeprecationAndroid{
-    "LinkDoctorDeprecationAndroid", base::FEATURE_DISABLED_BY_DEFAULT};
+    "LinkDoctorDeprecationAndroid", base::FEATURE_ENABLED_BY_DEFAULT};
 #endif  // defined(OS_ANDROID)
 
 // Enables LiteVideos, a data-saving optimization that throttles media requests
-- 
2.33.0

