From 098d0f0fa190e0648399b00cdac05318dcb0877a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:46:41 -0400
Subject: [PATCH 63/80] remove navigation error preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java    | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index aa5a7e1061d0..47fc4dd49c94 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -199,13 +199,11 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         mSearchSuggestions.setOnPreferenceChangeListener(this);
         mSearchSuggestions.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
 
-        mNavigationError = (ChromeSwitchPreference) findPreference(PREF_NAVIGATION_ERROR);
-        mNavigationError.setOnPreferenceChangeListener(this);
-        mNavigationError.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
-
         PreferenceCategory servicesCategory =
                 (PreferenceCategory) findPreference(PREF_SERVICES_CATEGORY);
 
+        removePreference(servicesCategory, findPreference(PREF_NAVIGATION_ERROR));
+
         // If the metrics-settings-android flag is not enabled, remove the corresponding element.
         if (!ChromeFeatureList.isEnabled(ChromeFeatureList.METRICS_SETTINGS_ANDROID)) {
             removePreference(servicesCategory, findPreference(PREF_METRICS_SETTINGS));
@@ -535,7 +533,7 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         //updateSyncPreferences();
 
         mSearchSuggestions.setChecked(mPrefService.getBoolean(Pref.SEARCH_SUGGEST_ENABLED));
-        mNavigationError.setChecked(mPrefService.getBoolean(Pref.ALTERNATE_ERROR_PAGES_ENABLED));
+        //mNavigationError.setChecked(mPrefService.getBoolean(Pref.ALTERNATE_ERROR_PAGES_ENABLED));
 
         mUsageAndCrashReporting.setChecked(
                 mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
-- 
2.32.0

