From aba6356a2b518630f04a7cd3ee8ff41f341f6ba0 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:49:13 -0400
Subject: [PATCH 64/79] remove usage and crash reports preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 47fc4dd49c94..fda6911088af 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -211,8 +211,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUsageAndCrashReporting =
                 (ChromeSwitchPreference) findPreference(PREF_USAGE_AND_CRASH_REPORTING);
-        mUsageAndCrashReporting.setOnPreferenceChangeListener(this);
-        mUsageAndCrashReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUsageAndCrashReporting.setOnPreferenceChangeListener(this);
+        //mUsageAndCrashReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUsageAndCrashReporting);
+        mUsageAndCrashReporting = null;
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
@@ -535,8 +537,8 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         mSearchSuggestions.setChecked(mPrefService.getBoolean(Pref.SEARCH_SUGGEST_ENABLED));
         //mNavigationError.setChecked(mPrefService.getBoolean(Pref.ALTERNATE_ERROR_PAGES_ENABLED));
 
-        mUsageAndCrashReporting.setChecked(
-                mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
+        //mUsageAndCrashReporting.setChecked(
+                //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
         mUrlKeyedAnonymizedData.setChecked(
                 UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
                         getProfile()));
-- 
2.32.0

