From 04e01e21fd9b36f8c245720bae5fb9eeb57c64ce Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 27 May 2021 07:30:02 -0400
Subject: [PATCH] add back trichrome_chrome_apk targets

---
 chrome/android/BUILD.gn | 46 ++++++++++++++++++++++++++++++++++++++++-
 1 file changed, 45 insertions(+), 1 deletion(-)

diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index c82bd7adebd57..407394d254b67 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -2273,6 +2273,10 @@ if (current_toolchain == default_toolchain) {
     is_monochrome = true
     is_bundle_module = true
   }
+  resource_packaging("trichrome_chrome_apk_pak_assets") {
+    is_monochrome = false
+    is_trichrome = true
+  }
   resource_packaging("trichrome_chrome_bundle_module_pak_assets") {
     is_monochrome = false
     is_trichrome = true
@@ -2831,7 +2835,7 @@ if (android_64bit_target_cpu && skip_secondary_abi_for_cq) {
 
     if (trichrome_synchronized_proguard) {
       webview_target = "//android_webview:trichrome_webview_apk"
-      chrome_target = ":trichrome_chrome_bundle"
+      chrome_target = ":trichrome_chrome_apk"
     }
 
     if (_enable_manifest_verification) {
@@ -2887,6 +2891,46 @@ android_resources("trichrome_dummy_resources") {
   sources = [ "trichrome/res_dummy/values/strings.xml" ]
 }
 
+monochrome_public_apk_or_module_tmpl("trichrome_chrome_apk") {
+  version_code = trichrome_version_code
+  version_name = chrome_version_name
+  apk_name = "TrichromeChrome"
+  target_type = "android_apk"
+  use_trichrome_library = true
+  static_library_provider = ":trichrome_library_apk"
+  if (!is_java_debug) {
+    static_library_synchronized_proguard = trichrome_synchronized_proguard
+    if (trichrome_synchronized_proguard) {
+      resource_ids_provider_dep = "//android_webview:trichrome_webview_apk"
+    }
+  }
+  if (android_64bit_target_cpu) {
+    is_64_bit_browser = false
+    include_64_bit_webview = true
+  }
+}
+
+if (android_64bit_target_cpu) {
+  monochrome_public_apk_or_module_tmpl("trichrome_chrome_64_32_apk") {
+    version_code = trichrome_version_code
+    version_name = chrome_version_name
+    apk_name = "TrichromeChrome6432"
+    target_type = "android_apk"
+    use_trichrome_library = true
+    static_library_provider = ":trichrome_library_apk"
+    if (!is_java_debug) {
+      static_library_synchronized_proguard = trichrome_synchronized_proguard
+      if (trichrome_synchronized_proguard) {
+        resource_ids_provider_dep = "//android_webview:trichrome_webview_apk"
+      }
+    }
+    if (android_64bit_target_cpu) {
+      is_64_bit_browser = true
+      include_32_bit_webview = true
+    }
+  }
+}
+
 chrome_public_unit_test_apk_manifest =
     "$root_gen_dir/chrome_public_unit_test_apk_manifest/AndroidManifest.xml"
 chrome_public_test_apk_manifest =
