From 402c26f7dbae2a93c76d751aaceae461132976b0 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 8 Apr 2020 09:04:51 -0400
Subject: [PATCH 64/77] remove password leak detection preference

---
 .../browser/sync/settings/SyncAndServicesSettings.java | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 43915274292e..3a4abd3e4c4b 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -240,8 +240,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
             mPasswordLeakDetection =
                     (ChromeSwitchPreference) findPreference(PREF_PASSWORD_LEAK_DETECTION);
-            mPasswordLeakDetection.setOnPreferenceChangeListener(this);
-            mPasswordLeakDetection.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+            //mPasswordLeakDetection.setOnPreferenceChangeListener(this);
+            //mPasswordLeakDetection.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+            removePreference(servicesCategory, mPasswordLeakDetection);
+            mPasswordLeakDetection = null;
 
             mSafeBrowsingReporting =
                     (ChromeSwitchPreference) findPreference(PREF_SAFE_BROWSING_SCOUT_REPORTING);
@@ -670,6 +672,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
         mSafeBrowsingReporting.setChecked(safe_browsing_enabled
                 && SafeBrowsingBridge.isSafeBrowsingExtendedReportingEnabled());
 
+        if (mPasswordLeakDetection == null) {
+            return;
+        }
+
         boolean has_token_for_leak_check = PasswordUIView.hasAccountForLeakCheckRequest();
         boolean leak_detection_enabled =
                 mPrefService.getBoolean(Pref.PASSWORD_LEAK_DETECTION_ENABLED);
-- 
2.31.1

