From 372d4d2e6487b0a4b3ea4b5915c0f611e6c8b448 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:48:41 -0400
Subject: [PATCH 65/79] remove safe browsing reporting preference

---
 .../sync/settings/SyncAndServicesSettings.java       | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index 3a4abd3e4c4b..1a77082e4130 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -247,8 +247,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
             mSafeBrowsingReporting =
                     (ChromeSwitchPreference) findPreference(PREF_SAFE_BROWSING_SCOUT_REPORTING);
-            mSafeBrowsingReporting.setOnPreferenceChangeListener(this);
-            mSafeBrowsingReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+            //mSafeBrowsingReporting.setOnPreferenceChangeListener(this);
+            //mSafeBrowsingReporting.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+            removePreference(servicesCategory, mSafeBrowsingReporting);
+            mSafeBrowsingReporting = null;
         }
 
         // If the metrics-settings-android flag is not enabled, remove the corresponding element.
@@ -668,9 +670,9 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
     private void updateLeakDetectionAndSafeBrowsingReportingPreferences() {
         assert !mIsSafeBrowsingPreferenceRemoved;
         boolean safe_browsing_enabled = mPrefService.getBoolean(Pref.SAFE_BROWSING_ENABLED);
-        mSafeBrowsingReporting.setEnabled(safe_browsing_enabled);
-        mSafeBrowsingReporting.setChecked(safe_browsing_enabled
-                && SafeBrowsingBridge.isSafeBrowsingExtendedReportingEnabled());
+        //mSafeBrowsingReporting.setEnabled(safe_browsing_enabled);
+        //mSafeBrowsingReporting.setChecked(safe_browsing_enabled
+                //&& SafeBrowsingBridge.isSafeBrowsingExtendedReportingEnabled());
 
         if (mPasswordLeakDetection == null) {
             return;
-- 
2.31.1

