From 1bf85d0c85e9d52f9796c8694a5e4dbdbbce7960 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 2 Aug 2019 23:50:03 -0400
Subject: [PATCH 65/78] remove url keyed anonymized data preference

---
 .../sync/settings/SyncAndServicesSettings.java       | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
index fda6911088af..fc83212c2df6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/sync/settings/SyncAndServicesSettings.java
@@ -218,8 +218,10 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         mUrlKeyedAnonymizedData =
                 (ChromeSwitchPreference) findPreference(PREF_URL_KEYED_ANONYMIZED_DATA);
-        mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
-        mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        //mUrlKeyedAnonymizedData.setOnPreferenceChangeListener(this);
+        //mUrlKeyedAnonymizedData.setManagedPreferenceDelegate(mManagedPreferenceDelegate);
+        removePreference(servicesCategory, mUrlKeyedAnonymizedData);
+        mUrlKeyedAnonymizedData = null;
 
         mAutofillAssistant = (ChromeSwitchPreference) findPreference(PREF_AUTOFILL_ASSISTANT);
         Preference autofillAssistantSubsection = findPreference(PREF_AUTOFILL_ASSISTANT_SUBSECTION);
@@ -539,9 +541,9 @@ public class SyncAndServicesSettings extends PreferenceFragmentCompat
 
         //mUsageAndCrashReporting.setChecked(
                 //mPrivacyPrefManager.isUsageAndCrashReportingPermittedByUser());
-        mUrlKeyedAnonymizedData.setChecked(
-                UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
-                        getProfile()));
+        //mUrlKeyedAnonymizedData.setChecked(
+                //UnifiedConsentServiceBridge.isUrlKeyedAnonymizedDataCollectionEnabled(
+                        //getProfile()));
 
         if (mAutofillAssistant != null) {
             mAutofillAssistant.setChecked(isAutofillAssistantSwitchOn());
-- 
2.31.1

