From 6cfc6d17526480afdc18c801fbe28de75f1f8360 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 21 Nov 2020 01:30:06 -0500
Subject: [PATCH 72/74] disable checkout_nacl

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index 337d0bd7d998..794b7fef18ab 100644
--- a/DEPS
+++ b/DEPS
@@ -94,7 +94,7 @@ vars = {
 
   # Check out and download nacl by default. This can be disabled e.g. with
   # custom_vars.
-  'checkout_nacl': True,
+  'checkout_nacl': False,
 
   # By default, do not check out src-internal. This can be overridden e.g. with
   # custom_vars.
-- 
2.29.2

