From 7f434588110190fd940bfa8e871e2eb649d22c98 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 21 Nov 2020 01:30:06 -0500
Subject: [PATCH 72/75] disable checkout_nacl

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index 84b8203a4cb4..0a30a2d108b1 100644
--- a/DEPS
+++ b/DEPS
@@ -94,7 +94,7 @@ vars = {
 
   # Check out and download nacl by default. This can be disabled e.g. with
   # custom_vars.
-  'checkout_nacl': True,
+  'checkout_nacl': False,
 
   # By default, do not check out src-internal. This can be overridden e.g. with
   # custom_vars.
-- 
2.29.2

