From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 23 Dec 2020 06:00:50 -0500
Subject: [PATCH] enable split cache by default

---
 net/base/features.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 58aa8a22e7e64..04b31805982a4 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -104,10 +104,10 @@ const base::FeatureParam<base::TimeDelta>
         &kNetworkQualityEstimator,
         "EffectiveConnectionTypeRecomputationInterval", base::Seconds(10)};
 
-BASE_FEATURE(SplitCacheByIncludeCredentials, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(SplitCacheByIncludeCredentials, base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(SplitCacheByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Note: Use of this feature is gated on the HTTP cache itself being
 // partitioned, which is controlled by the kSplitCacheByNetworkIsolationKey
@@ -116,7 +116,7 @@ BASE_FEATURE(SplitCacheByCrossSiteMainFrameNavigationBoolean,
              base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(SplitCodeCacheByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(PartitionConnectionsByNetworkIsolationKey,
              base::FEATURE_DISABLED_BY_DEFAULT);
