From cbd603102ae632ca7039e7384eef8d0a55f3a2e9 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Fri, 15 Oct 2021 13:49:33 +0000
Subject: [PATCH 77/78] Remove privacy trials preference

---
 .../chrome/browser/privacy/settings/PrivacySettings.java        | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
index c0d10a607cddc..dad9f2d25ddbd 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
@@ -100,6 +100,8 @@ public class PrivacySettings
 
         setUpIncognitoReauthPreference();
 
+        getPreferenceScreen().removePreference(findPreference(PREF_PRIVACY_SANDBOX));
+
         Preference safeBrowsingPreference = findPreference(PREF_SAFE_BROWSING);
         safeBrowsingPreference.setSummary(
                 SafeBrowsingSettingsFragment.getSafeBrowsingSummaryString(getContext()));
-- 
2.34.1

