From e47e68d70aa9ece1ed06189953bc391aa8a6560c Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Mon, 24 Jan 2022 19:22:26 +0000
Subject: [PATCH] Hide currently no-op preload settings

---
 .../chrome/browser/privacy/settings/PrivacySettings.java         | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
index 4728e4f9f26de..f490904bd0ed6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
@@ -259,6 +259,7 @@ public class PrivacySettings
         if (preloadPagesPreference != null) {
             preloadPagesPreference.setSummary(
                     PreloadPagesSettingsFragment.getPreloadPagesSummaryString(getContext()));
+            getPreferenceScreen().removePreference(preloadPagesPreference);
         }
 
         Preference secureDnsPref = findPreference(PREF_SECURE_DNS);
