From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 15 Apr 2021 02:14:37 -0400
Subject: [PATCH] remove Help & feedback menu entry

---
 .../browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
index 6d8ef314c1f13..e533aed2a3801 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
@@ -414,7 +414,7 @@ public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateIm
         }
 
         // Help
-        modelList.add(buildHelpItem());
+        // modelList.add(buildHelpItem());
 
         // Managed by
         if (shouldShowManagedByMenuItem(currentTab)) {
