From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 15 Apr 2021 02:14:37 -0400
Subject: [PATCH] remove Help & feedback menu entry

---
 .../browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java    | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
index 8c8b7c60395e8..5c8199ebdfaab 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
@@ -346,6 +346,8 @@ public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateIm
         menu.findItem(R.id.reader_mode_menu_id)
                 .setVisible(DomDistillerFeatures.showAlwaysOnEntryPoint());
 
+        menu.findItem(R.id.help_id).setVisible(false);
+
         updateManagedByMenuItem(menu, currentTab);
 
         // Only display quick delete divider line on the regular mode page menu.
