From 29f79222338f8381881b2e0f447dc496de48b959 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Mon, 24 Jan 2022 19:22:26 +0000
Subject: [PATCH] Hide currently no-op preload settings

---
 .../chrome/browser/privacy/settings/PrivacySettings.java         | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
index d446913f94a97..4f7fc2d185063 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java
@@ -257,6 +257,7 @@ public class PrivacySettings
         if (preloadPagesPreference != null) {
             preloadPagesPreference.setSummary(
                     PreloadPagesSettingsFragment.getPreloadPagesSummaryString(getContext()));
+            getPreferenceScreen().removePreference(preloadPagesPreference);
         }
 
         Preference secureDnsPref = findPreference(PREF_SECURE_DNS);
