From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 17 Jun 2019 13:14:22 -0400
Subject: [PATCH] always use local new tab page

---
 chrome/browser/search/search.cc | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/chrome/browser/search/search.cc b/chrome/browser/search/search.cc
index 6d3e318b62f14..5a68151214644 100644
--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -163,6 +163,10 @@ struct NewTabURLDetails {
   NewTabURLDetails(const GURL& url, NewTabURLState state)
       : url(url), state(state) {}
 
+  static bool ShouldUseLocalNewTab() {
+    return true;
+  }
+
   static NewTabURLDetails ForProfile(Profile* profile) {
     // Incognito and Guest profiles have their own New Tab.
     // This function may also be called by other off-the-record profiles that
@@ -178,6 +182,9 @@ struct NewTabURLDetails {
     const bool is_desktop_android = IsWebUiNtpEnabledForDesktopAndroid();
     const GURL local_url(
         is_desktop_android ? chrome::ChromeUINewTabPageURLAsGURL() : GURL());
+    if (ShouldUseLocalNewTab()) {
+      return NewTabURLDetails(local_url, NEW_TAB_URL_VALID);
+    }
     if (is_desktop_android) {
       if (default_is_google) {
         return NewTabURLDetails(local_url, NEW_TAB_URL_VALID);
