From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Sat, 16 Dec 2023 00:25:55 +0000
Subject: [PATCH] Remove privacy guides preference UI

---
 .../chrome/browser/privacy/settings/PrivacySettingsExt.java     | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java
index ad71ef73656bf..e7d66688e13da 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java
@@ -56,6 +56,8 @@ final class PrivacySettingsExt {
                 prefFragment, PrivacySettings.PREF_SYNC_AND_SERVICES_LINK);
         SettingsExtUtils.safelyRemovePreference(
                 prefFragment, PrivacySettings.PREF_PRIVACY_SANDBOX);
+        SettingsExtUtils.safelyRemovePreference(
+                prefFragment, PrivacySettings.PREF_PRIVACY_GUIDE);
     }
 
     static void initializePreferences(@NonNull PreferenceFragmentCompat prefFragment, @NonNull Profile profile) {
