From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: OpSecJunkrat <admin@nullnet.services>
Date: Mon, 29 May 2023 07:19:45 -0400
Subject: [PATCH] Capitalize JavaScript as a trademark all the time

---
 /components/site_settings_strings.grdp   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/site_settings_strings.grdp b/components/site_settings_strings.grdp
index efc94cd..8277f0c 100644
--- a/components/site_settings_strings.grdp
+++ b/components/site_settings_strings.grdp
@@ -80,7 +80,7 @@
     JavaScript
   </message>
   <message name="IDS_SITE_SETTINGS_TYPE_JAVASCRIPT_MID_SENTENCE" desc="The label used for JavaScript site settings controls when used mid-sentence.">
-    javascript
+    JavaScript
   </message>
   <message name="IDS_SITE_SETTINGS_TYPE_LOCATION" desc="The label used for geolocation site settings controls." meaning="Geolocation">
     Location
