From 793e745a391a90d1126b1e240a07f759513aa39f Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Sat, 15 Apr 2023 04:09:43 +0000
Subject: [PATCH] Enable partitioned cookies by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 7f68e7f6f5701..31d8f16fcca3c 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -237,7 +237,7 @@ BASE_FEATURE(kSamePartyAttributeEnabled,
 
 BASE_FEATURE(kPartitionedCookies,
              "PartitionedCookies",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNoncedPartitionedCookies,
              "NoncedPartitionedCookies",
