From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <daniel.micay@grapheneos.org>
Date: Fri, 17 Nov 2023 03:38:17 -0500
Subject: [PATCH] enable checkout_pgo_profiles by default

---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index ba8384206c8b7..49de28c00aff0 100644
--- a/DEPS
+++ b/DEPS
@@ -150,7 +150,7 @@ vars = {
   'checkout_clang_coverage_tools': False,
 
   # Fetch the pgo profiles to optimize official builds.
-  'checkout_pgo_profiles': False,
+  'checkout_pgo_profiles': True,
 
   # Fetch clang-tidy into the same bin/ directory as our clang binary.
   'checkout_clang_tidy': False,
