From c77ca6ee9b5bbeadd7fe8a8a8eb160bf193cfbb3 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Sat, 15 Apr 2023 04:09:43 +0000
Subject: [PATCH] Enable partitioned cookies by default

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 06e6afc45a4fc..a66997561da4a 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -241,7 +241,7 @@ BASE_FEATURE(kSamePartyAttributeEnabled,
 
 BASE_FEATURE(kPartitionedCookies,
              "PartitionedCookies",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNoncedPartitionedCookies,
              "NoncedPartitionedCookies",
