From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 15 Apr 2021 02:14:37 -0400
Subject: [PATCH] remove Help & feedback menu entry

---
 .../tabbed_mode/TabbedAppMenuPropertiesDelegate.java   | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
index 6a9467d96a630..7c748cbe72a55 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedAppMenuPropertiesDelegate.java
@@ -553,7 +553,10 @@ public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateIm
         }
 
         // Help
-        modelList.add(buildHelpItem());
+        boolean enableHelpItem = false;
+        if (enableHelpItem) {
+            modelList.add(buildHelpItem());
+        }
 
         // Managed by
         if (shouldShowManagedByMenuItem(currentTab)) {
@@ -739,7 +742,10 @@ public class TabbedAppMenuPropertiesDelegate extends AppMenuPropertiesDelegateIm
         modelList.add(buildSettingsItem());
 
         // Help
-        modelList.add(buildHelpParentItem());
+        boolean enableHelpItem = false;
+        if (enableHelpItem) {
+            modelList.add(buildHelpParentItem());
+        }
 
         // Managed by
         if (shouldShowManagedByMenuItem(currentTab)) {
