From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <daniel.micay@grapheneos.org>
Date: Sat, 27 Apr 2024 13:17:05 -0400
Subject: [PATCH] enable hybrid post-quantum key exchange on Android

---
 net/base/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/base/features.cc b/net/base/features.cc
index 778fe01816534..cc45b57e4d962 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -142,7 +142,7 @@ BASE_FEATURE(kTLS13KeyUpdate,
 
 BASE_FEATURE(kPostQuantumKyber,
              "PostQuantumKyber",
-#if BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_IOS)
+#if BUILDFLAG(IS_IOS)
              base::FEATURE_DISABLED_BY_DEFAULT);
 #else
              base::FEATURE_ENABLED_BY_DEFAULT);
