From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <daniel.micay@grapheneos.org>
Date: Thu, 16 Jan 2025 15:40:35 -0500
Subject: [PATCH] remove debug dump when MTE is disabled

---
 base/allocator/partition_alloc_support.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/base/allocator/partition_alloc_support.cc b/base/allocator/partition_alloc_support.cc
index 9d154b45d8352..33eca61e32f5d 100644
--- a/base/allocator/partition_alloc_support.cc
+++ b/base/allocator/partition_alloc_support.cc
@@ -1197,7 +1197,6 @@ void PartitionAllocSupport::ReconfigureAfterFeatureListInit(
         LOG(ERROR) << "PartitionAlloc: Failed to enable memory tagging due to "
                       "MTE disabled at startup (Process: "
                    << process_type << ")";
-        debug::DumpWithoutCrashing();
         enable_memory_tagging = false;
       }
 
