From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Thu, 22 Aug 2024 13:25:50 +0000
Subject: [PATCH] Enable blocking cross partition blob url fetching by default

---
 storage/browser/blob/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/storage/browser/blob/features.cc b/storage/browser/blob/features.cc
index 17003b52f595c..627acffc08bbc 100644
--- a/storage/browser/blob/features.cc
+++ b/storage/browser/blob/features.cc
@@ -9,7 +9,7 @@ namespace features {
 // Please keep features in alphabetical order.
 BASE_FEATURE(kBlockCrossPartitionBlobUrlFetching,
              "BlockCrossPartitionBlobUrlFetching",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 // Please keep features in alphabetical order.
 
