From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Thu, 12 Dec 2024 07:09:40 +0000
Subject: [PATCH] Reduce user agent and client hints for webview

---
 .../vanadium/config/host/ConfigGenerator.java   | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
index 84ffa5ba4a43d..4019a0ab32d9e 100644
--- a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
+++ b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
@@ -44,6 +44,23 @@ public class ConfigGenerator {
                         )
                 ))
         ));
+        configList.add(config(configParams -> configParams.setSpec(
+                        spec(specParams -> specParams.setSpecTypes(getSpecTypes(SpecType.WEBVIEW))
+                        ))
+                .addAllFlags(flags(
+                        flag(flagParams -> flagParams.setFlagName("ReduceUserAgentMinorVersion")
+                                .setFlagType(FlagType.ENABLED_FEATURE)
+                        )
+                        ,
+                        flag(flagParams -> flagParams.setFlagName("WebViewReduceUAAndroidVersionDeviceModel")
+                                .setFlagType(FlagType.ENABLED_FEATURE)
+                        )
+                        ,
+                        flag(flagParams -> flagParams.setFlagName("ClientHintsFromReducedUA")
+                                .setFlagType(FlagType.ENABLED_FEATURE)
+                        )
+                ))
+        ));
         return sortConfigs(configList);
     }
 
