From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Tue, 13 May 2025 13:53:31 +0000
Subject: [PATCH] Hide potentially no-op password leak detection toggle

---
 .../chrome/browser/privacy/settings/PrivacySettingsExt.java     | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java
index 6c287e0c17852..1475014a8f595 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettingsExt.java
@@ -79,6 +79,8 @@ final class PrivacySettingsExt {
                 prefFragment, PrivacySettings.PREF_PRIVACY_GUIDE);
         SettingsExtUtils.safelyRemovePreference(
                 prefFragment, PrivacySettings.PREF_JAVASCRIPT_OPTIMIZER);
+        SettingsExtUtils.safelyRemovePreference(
+                prefFragment, PrivacySettings.PREF_PASSWORD_LEAK_DETECTION);
     }
 
     static void initializePreferences(@NonNull PreferenceFragmentCompat prefFragment, @NonNull Profile profile) {
