From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Thu, 24 Jul 2025 16:04:26 +0000
Subject: [PATCH] hide and don't fetch icons for search engine settings

---
 .../settings/SearchEngineAdapter.java         | 37 ++++++++++++++++++-
 1 file changed, 36 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/search_engines/android/java/src/org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter.java b/chrome/browser/search_engines/android/java/src/org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter.java
index 0cda3cbdd91a5..aca116545ecbd 100644
--- a/chrome/browser/search_engines/android/java/src/org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter.java
+++ b/chrome/browser/search_engines/android/java/src/org/chromium/chrome/browser/search_engines/settings/SearchEngineAdapter.java
@@ -540,7 +540,42 @@ public class SearchEngineAdapter extends BaseAdapter
                         templateUrlService.getSearchEngineUrlFromTemplateUrl(
                                 templateUrl.getKeyword()));
 
-        updateLogo(logoView, templateUrl, faviconUrl);
+
+        if (logoView != null) {
+            boolean shouldHideLogo = true;
+            if (shouldHideLogo) {
+                var logoLayoutParams = logoView.getLayoutParams();
+                final int logoMarginStart;
+                if (logoLayoutParams instanceof
+                        android.widget.LinearLayout.LayoutParams logoMarginLayoutParams) {
+                    logoMarginStart = logoMarginLayoutParams.getMarginStart();
+                } else {
+                    logoMarginStart = Integer.MIN_VALUE;
+                }
+
+                if (logoMarginStart != Integer.MIN_VALUE) {
+                    if (description.getVisibility() != View.GONE) {
+                        var layoutParams = description.getLayoutParams();
+                        if (layoutParams instanceof
+                                android.widget.LinearLayout.LayoutParams marginLayoutParams) {
+                            marginLayoutParams.setMarginStart(logoMarginStart);
+                            description.setLayoutParams(marginLayoutParams);
+                        }
+                    }
+                    if (url.getVisibility() != View.GONE) {
+                        var layoutParams = url.getLayoutParams();
+                        if (layoutParams instanceof
+                                android.widget.LinearLayout.LayoutParams marginLayoutParams) {
+                            marginLayoutParams.setMarginStart(logoMarginStart);
+                            url.setLayoutParams(marginLayoutParams);
+                        }
+                    }
+                }
+                logoView.setVisibility(View.GONE);
+            } else {
+                updateLogo(logoView, templateUrl, faviconUrl);
+            }
+        }
 
         // To improve the explore-by-touch experience, the radio button is hidden from accessibility
         // and instead, "checked" or "not checked" is read along with the search engine's name, e.g.
