From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <daniel.micay@grapheneos.org>
Date: Wed, 26 Feb 2025 17:04:57 -0500
Subject: [PATCH] fix configuration error

---
 chrome/android/trichrome.gni | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/android/trichrome.gni b/chrome/android/trichrome.gni
index fefc42a71aed7..efe781f0449fb 100644
--- a/chrome/android/trichrome.gni
+++ b/chrome/android/trichrome.gni
@@ -56,6 +56,7 @@ template("trichrome_library_apk_tmpl") {
   if (!_include_primary_abi) {
     _secondary_out_dir =
         get_label_info("X($android_secondary_abi_toolchain)", "root_out_dir")
+    not_needed([ "_secondary_out_dir" ])
   }
 
   if (defined(invoker.manifest_package)) {
