From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Tue, 25 Mar 2025 09:17:53 +0000
Subject: [PATCH] Fix versionCode for 64-bit only builds

---
 chrome/version.gni | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/chrome/version.gni b/chrome/version.gni
index 3292823aee62b..2632f784fa807 100644
--- a/chrome/version.gni
+++ b/chrome/version.gni
@@ -325,9 +325,9 @@ if (is_android) {
         }
       } else {
         if (_use_stable_package_name_for_trichrome) {
-          true_true_true_false = trichrome_beta_version_code
+          true_true_true_false = trichrome_64_beta_version_code
         } else {
-          true_true_true_false = trichrome_version_code
+          true_true_true_false = trichrome_64_version_code
         }
       }
     } else {
@@ -349,7 +349,7 @@ if (is_android) {
         true_true_true_false = monochrome_64_version_code
         true_false_false_true = monochrome_32_version_code
       } else {
-        true_true_true_false = monochrome_version_code
+        true_true_true_false = monochrome_64_version_code
       }
     } else {
       false_false_false_true = monochrome_version_code
@@ -376,11 +376,11 @@ if (is_android) {
         }
       } else {
         if (android_channel == "dev") {
-          true_true_false = webview_dev_version_code
+          true_true_false = webview_64_dev_version_code
         } else if (android_channel == "beta") {
-          true_true_false = webview_beta_version_code
+          true_true_false = webview_64_beta_version_code
         } else {
-          true_true_false = webview_stable_version_code
+          true_true_false = webview_64_stable_version_code
         }
       }
     } else {
