From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Sat, 3 May 2025 16:48:48 +0000
Subject: [PATCH] tmp: config: disable optimization services features

---
 .../vanadium/config/host/ConfigGenerator.java | 26 +++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
index 78858f252df43..c0fff2c141ba4 100644
--- a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
+++ b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
@@ -90,6 +90,32 @@ public class ConfigGenerator {
                         )
                 ))
         ));
+        configList.add(config(configParams -> configParams.setSpec(
+                        spec(specParams -> specParams.setSpecTypes(getSpecTypes(SpecType.BROWSER))
+                                .setMaxVersion(7103_087_99L)
+                        ))
+                .addAllFlags(flags(
+                        flag(flagParams -> flagParams.setFlagName("OptimizationHints")
+                                .setFlagType(FlagType.DISABLED_FEATURE)
+                        )
+                        ,
+                        flag(flagParams -> flagParams.setFlagName("OptimizationHintsFetchingAnonymousDataConsent")
+                                .setFlagType(FlagType.DISABLED_FEATURE)
+                        )
+                        ,
+                        flag(flagParams -> flagParams.setFlagName("OptimizationGuideModelDownloading")
+                                .setFlagType(FlagType.DISABLED_FEATURE)
+                        )
+                        ,
+                        flag(flagParams -> flagParams.setFlagName("OptimizationPersonalizedHintsFetching")
+                                .setFlagType(FlagType.DISABLED_FEATURE)
+                        )
+                        ,
+                        flag(flagParams -> flagParams.setFlagName("TextSafetyClassifier")
+                                .setFlagType(FlagType.DISABLED_FEATURE)
+                        )
+                ))
+        ));
         return sortConfigs(configList);
     }
 
