From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Tue, 13 May 2025 14:59:01 +0000
Subject: [PATCH] Definition of Mullvad Leta type of search engine

---
 components/search_engines/search_engine_type.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/search_engines/search_engine_type.h b/components/search_engines/search_engine_type.h
index e1228cbf36a79..07d1f39164d59 100644
--- a/components/search_engines/search_engine_type.h
+++ b/components/search_engines/search_engine_type.h
@@ -98,6 +98,7 @@ enum SearchEngineType {
   SEARCH_ENGINE_FREESPOKE = 78,
   SEARCH_ENGINE_KAGI = 79,
   SEARCH_ENGINE_STARTER_PACK_AI_MODE = 80,
+  SEARCH_ENGINE_MULLVAD_LETA = 81,
 
   SEARCH_ENGINE_MAX  // Bounding value needed for UMA histogram macro.
 };
