From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <88831734+quh4gko8@users.noreply.github.com>
Date: Mon, 9 Jun 2025 06:15:30 +0000
Subject: [PATCH] Disable permission prompt for local network access in Android

---
 .../browser/permissions/permission_controller_impl.cc | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/content/browser/permissions/permission_controller_impl.cc b/content/browser/permissions/permission_controller_impl.cc
index f53158596f935..91aad967e03c2 100644
--- a/content/browser/permissions/permission_controller_impl.cc
+++ b/content/browser/permissions/permission_controller_impl.cc
@@ -159,6 +159,17 @@ bool IsRequestAllowed(
   for (const auto& permission : permissions) {
     PermissionType permission_type =
         blink::PermissionDescriptorToPermissionType(permission);
+#if BUILDFLAG(IS_ANDROID)
+    if (permission_type == blink::PermissionType::LOCAL_NETWORK_ACCESS) {
+      render_frame_host->GetOutermostMainFrame()->AddMessageToConsole(
+          blink::mojom::ConsoleMessageLevel::kWarning,
+          blink::GetPermissionString(permission_type) +
+              " permission has been blocked because it was requested "
+              "in Android, which is currently unsupported.");
+      is_permission_allowed = false;
+      continue;
+    }
+#endif // BUILDFLAG(IS_ANDROID)
     PermissionResult result =
         VerifyContextOfCurrentDocument(permission_type, render_frame_host);
 
