From 98c0f34dae38b0f79927bc476eb070cc74dec21c Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Sat, 10 May 2025 01:30:54 +0000
Subject: [PATCH] config: enable wasm in interpreter mode via DrumBrake

---
 .../app/vanadium/config/host/ConfigGenerator.java     | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
index c75f23124e5cc..933e4652d5855 100644
--- a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
+++ b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
@@ -51,6 +51,17 @@ public class ConfigGenerator {
                         )
                 ))
         ));
+        configList.add(config(configParams -> configParams.setSpec(
+                        spec(specParams -> specParams.setSpecTypes(getSpecTypes(SpecType.BROWSER, SpecType.WEBVIEW))
+                        ))
+                .addAllFlags(flags(
+                        flag(flagParams -> flagParams.setFlagName("js-flags")
+                                .addFlagValue("--jitless")
+                                .addFlagValue("--wasm-jitless")
+                                .setFlagType(FlagType.ENABLED_SWITCH)
+                        )
+                ))
+        ));
         return sortConfigs(configList);
     }
 
