From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Sat, 10 May 2025 01:30:54 +0000
Subject: [PATCH] tmp: config: enable origin keyed process isolation by default

---
 .../app/vanadium/config/host/ConfigGenerator.java     | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
index ece34de1d8f1d..d1b198f9e1aba 100644
--- a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
+++ b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
@@ -62,6 +62,17 @@ public class ConfigGenerator {
                         )
                 ))
         ));
+        configList.add(config(configParams -> configParams.setSpec(
+                        spec(specParams -> specParams.setSpecTypes(getSpecTypes(SpecType.BROWSER, SpecType.WEBVIEW))
+                                .setMinVersion(7390_070_00L)
+                                .setMaxVersion(7390_070_99L)
+                        ))
+                .addAllFlags(flags(
+                        flag(flagParams -> flagParams.setFlagName("OriginKeyedProcessesByDefault") // prerequisite feature string "OriginAgentClusterDefaultEnable" already enabled by default upstream.
+                                .setFlagType(FlagType.ENABLED_FEATURE)
+                        )
+                ))
+        ));
         return sortConfigs(configList);
     }
 
