From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Fri, 7 Nov 2025 06:29:43 +0000
Subject: [PATCH] disable misleading new tab cards by default

---
 .../magic_stack/HomeModulesConfigManager.java    | 16 ++++++----------
 1 file changed, 6 insertions(+), 10 deletions(-)

diff --git a/chrome/browser/magic_stack/android/java/src/org/chromium/chrome/browser/magic_stack/HomeModulesConfigManager.java b/chrome/browser/magic_stack/android/java/src/org/chromium/chrome/browser/magic_stack/HomeModulesConfigManager.java
index 0d80babfbdfb8..7b7cae55be620 100644
--- a/chrome/browser/magic_stack/android/java/src/org/chromium/chrome/browser/magic_stack/HomeModulesConfigManager.java
+++ b/chrome/browser/magic_stack/android/java/src/org/chromium/chrome/browser/magic_stack/HomeModulesConfigManager.java
@@ -86,7 +86,8 @@ public class HomeModulesConfigManager {
      * @param moduleType {@link ModuleType} needed to be notified to the listeners.
      */
     public boolean getPrefModuleTypeEnabled(@ModuleType int moduleType) {
-        return mSharedPreferencesManager.readBoolean(getSettingsPreferenceKey(moduleType), true);
+        // On Android, the magic stack is disabled by default,
+        return false;
     }
 
     /**
@@ -96,14 +97,13 @@ public class HomeModulesConfigManager {
      * @param enabled True is the module type is enabled.
      */
     public void setPrefModuleTypeEnabled(@ModuleType int moduleType, boolean enabled) {
-        mSharedPreferencesManager.writeBoolean(getSettingsPreferenceKey(moduleType), enabled);
-        notifyModuleTypeUpdated(moduleType, enabled);
+        // On Android, the magic stack is disabled by default,
     }
 
     /** Returns the user preference for whether all cards in the magic stack are enabled. */
     public boolean getPrefAllCardsEnabled() {
-        return mSharedPreferencesManager.readBoolean(
-                ChromePreferenceKeys.HOME_MODULE_CARDS_ENABLED, true);
+        // On Android, the magic stack is disabled by default,
+        return false;
     }
 
     /**
@@ -112,11 +112,7 @@ public class HomeModulesConfigManager {
      * @param enabled True is all cards are enabled.
      */
     public void setPrefAllCardsEnabled(boolean enabled) {
-        mSharedPreferencesManager.writeBoolean(
-                ChromePreferenceKeys.HOME_MODULE_CARDS_ENABLED, enabled);
-        for (HomeModulesStateListener listener : mHomepageStateListeners) {
-            listener.allCardsConfigChanged(enabled);
-        }
+        // On Android, the magic stack is disabled by default,
     }
 
     /** Sets a mocked instance for testing. */
