From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <88831734+quh4gko8@users.noreply.github.com>
Date: Mon, 1 Dec 2025 16:48:41 +0000
Subject: [PATCH] remove unused separate autofill status

---
 .../chrome/browser/autofill/AutofillClientProviderUtils.java | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java b/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java
index b147a6f5395f6..27992c5df5bcc 100644
--- a/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java
+++ b/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java
@@ -37,8 +37,6 @@ public class AutofillClientProviderUtils {
             "autofill_options_deep_link_shared_prefs_file";
     public static final String AUTOFILL_OPTIONS_DEEP_LINK_FEATURE_KEY =
             "AUTOFILL_OPTIONS_DEEP_LINK_FEATURE_KEY";
-    private static final String AWG_COMPONENT_NAME =
-            "com.google.android.gms/com.google.android.gms.autofill.service.AutofillService";
     private static @Nullable Integer sAndroidAutofillFrameworkAvailabilityForTesting;
 
     /**
@@ -98,9 +96,6 @@ public class AutofillClientProviderUtils {
         if (componentName == null) {
             return AndroidAutofillAvailabilityStatus.UNKNOWN_ANDROID_AUTOFILL_SERVICE;
         }
-        if (AWG_COMPONENT_NAME.equals(componentName.flattenToString())) {
-            return AndroidAutofillAvailabilityStatus.ANDROID_AUTOFILL_SERVICE_IS_GOOGLE;
-        }
         if (!prefs.getBoolean(Pref.AUTOFILL_USING_VIRTUAL_VIEW_STRUCTURE)) {
             return AndroidAutofillAvailabilityStatus.SETTING_TURNED_OFF;
         }
