From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <88831734+quh4gko8@users.noreply.github.com>
Date: Sat, 17 Jan 2026 17:08:00 +0000
Subject: [PATCH] config: disable unsupported HTTPS-first dialog UI by default

---
 .../app/vanadium/config/host/ConfigGenerator.java      | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
index b889e35b69442..a3b609cd2079a 100644
--- a/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
+++ b/vanadium/android_config/proto/java/src_host/app/vanadium/config/host/ConfigGenerator.java
@@ -102,6 +102,16 @@ public class ConfigGenerator {
                         )
                 ))
         ));
+        configList.add(config(configParams -> configParams.setSpec(
+                        spec(specParams -> specParams.setSpecTypes(getSpecTypes(SpecType.BROWSER, SpecType.WEBVIEW))
+                                .setMaxVersion(7559_077_00L)
+                        ))
+                .addAllFlags(flags(
+                        flag(flagParams -> flagParams.setFlagName("HttpsFirstDialogUi")
+                                .setFlagType(FlagType.DISABLED_FEATURE)
+                        )
+                ))
+        ));
         return sortConfigs(configList);
     }
 
