From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: inthewaves <inthewaves@pm.me>
Date: Tue, 9 Jun 2026 18:24:58 -0700
Subject: [PATCH] config: test: Add ConfigInfo on webview_instrumentation_apk

---
 android_webview/test/shared.gni | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/android_webview/test/shared.gni b/android_webview/test/shared.gni
index 22e43f3359fae..50c4c36650fd8 100644
--- a/android_webview/test/shared.gni
+++ b/android_webview/test/shared.gni
@@ -54,7 +54,10 @@ template("base_webview_instrumentation_apk") {
 
     shared_libraries =
         [ "//android_webview/test:libstandalonelibwebviewchromium" ]
-    srcjar_deps = [ "//android_webview/test:libstandalonelibwebviewchromium__jni_registration" ]
+    srcjar_deps = [
+      "//android_webview/test:libstandalonelibwebviewchromium__jni_registration",
+      "//vanadium/android_config:configinfo_srcjar_apk",
+    ]
 
     command_line_flags_file = "android-webview-command-line"
   }
