From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Wed, 12 Aug 2026 15:37:01 +0000
Subject: [PATCH] fixup! Use ruleset from config apk whenever it's present

---
 components/subresource_filter/content/browser/DEPS | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/components/subresource_filter/content/browser/DEPS b/components/subresource_filter/content/browser/DEPS
index bfed2dde27b25..06915d319c940 100644
--- a/components/subresource_filter/content/browser/DEPS
+++ b/components/subresource_filter/content/browser/DEPS
@@ -20,4 +20,6 @@ include_rules = [
   "+services/metrics/public/cpp",
   "+services/network/public/mojom",
   "+ui/base",
+  # Vanadium-specific dependencies
+  "+vanadium/chromium_src/components/subresource_filter/ext/android",
 ]
