From b9fbe6d8eeacafe88b75b56d377ebf24979513b9 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 18 Nov 2016 07:29:13 -0500
Subject: [PATCH 01/15] switch to -fstack-protector-strong

---
 build/config/compiler/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 1584926a007a..0f1f50227fea 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -207,7 +207,7 @@ config("compiler") {
       if (is_android && current_cpu == "x86") {
         cflags += [ "-fno-stack-protector" ]
       } else {
-        cflags += [ "-fstack-protector" ]
+        cflags += [ "-fstack-protector-strong" ]
       }
     }
 
-- 
2.11.0

