From f5274fa9843a8a431ad1840c3303c97f49d5da54 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 18 Nov 2016 07:29:13 -0500
Subject: [PATCH 01/16] switch to -fstack-protector-strong

---
 build/config/compiler/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index c4d61e2281a7..d97c11a70cbb 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -204,7 +204,7 @@ config("compiler") {
       if (is_android && current_cpu == "x86") {
         cflags += [ "-fno-stack-protector" ]
       } else {
-        cflags += [ "-fstack-protector" ]
+        cflags += [ "-fstack-protector-strong" ]
       }
     }
 
-- 
2.11.0

