From 0f51cd563e18ef36d8f05d7eb2850b1d74255b14 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 1 May 2019 11:39:59 -0400
Subject: [PATCH 02/45] use 64-bit Monochrome processes

---
 chrome/android/BUILD.gn | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 17311184c478..cdb97d71d90c 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -1844,6 +1844,13 @@ monochrome_public_apk_or_module_tmpl("monochrome_public_apk") {
   version_name = chrome_version_name
   apk_name = "MonochromePublic"
   target_type = "android_apk"
+
+  if (android_64bit_target_cpu) {
+    is_64_bit_browser = true
+    if (build_apk_secondary_abi) {
+      include_32_bit_webview = true
+    }
+  }
 }
 
 if (public_android_sdk) {
-- 
2.22.0

