From e40ef63756a04604284b33e87b3f40394f083935 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 26 Dec 2018 10:20:24 -0500
Subject: [PATCH 03/19] switch to -fstack-protector-strong

---
 build/config/compiler/BUILD.gn | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 87968b38e005..3b43ad83c37b 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -258,16 +258,12 @@ config("compiler") {
         cflags += [ "-fstack-protector" ]
       }
     } else if ((is_posix && !is_chromeos && !is_nacl) || is_fuchsia) {
-      # TODO(phajdan.jr): Use -fstack-protector-strong when our gcc supports it.
-      # See also https://crbug.com/533294
-      cflags += [ "--param=ssp-buffer-size=4" ]
-
       # The x86 toolchain currently has problems with stack-protector.
       if (is_android && current_cpu == "x86") {
         cflags += [ "-fno-stack-protector" ]
       } else if (current_os != "aix") {
         # Not available on aix.
-        cflags += [ "-fstack-protector" ]
+        cflags += [ "-fstack-protector-strong" ]
       }
     }
 
-- 
2.20.1

