From 59f32a2b8a080738764e51442299c44113a0dbef Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 22 Nov 2016 07:34:45 -0500
Subject: [PATCH 04/16] always use the WebView sandbox

---
 .../android/webview/chromium/WebViewChromiumFactoryProvider.java  | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
index 334b39fb4b13..9b2920dac97b 100644
--- a/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
+++ b/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromiumFactoryProvider.java
@@ -222,12 +222,8 @@ public class WebViewChromiumFactoryProvider implements WebViewFactoryProvider {
             CommandLine.init(null);
         }
 
-        if (Settings.Global.getInt(ContextUtils.getApplicationContext().getContentResolver(),
-                    Settings.Global.WEBVIEW_MULTIPROCESS, 0)
-                == 1) {
-            CommandLine cl = CommandLine.getInstance();
-            cl.appendSwitch("webview-sandboxed-renderer");
-        }
+        CommandLine cl = CommandLine.getInstance();
+        cl.appendSwitch("webview-sandboxed-renderer");
 
         ThreadUtils.setWillOverrideUiThread();
         // Load chromium library.
-- 
2.11.0

