From 378607920cc2fad1191c07647b0334c5ddb48b2a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sun, 13 Aug 2017 19:33:04 -0400
Subject: [PATCH 07/48] hide passwords.google.com link when not supported

---
 .../preferences/password/SavePasswordsPreferences.java        | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java b/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java
index 1142f3f36600..9ff0b8188611 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java
@@ -34,6 +34,7 @@ import org.chromium.chrome.browser.preferences.PrefServiceBridge;
 import org.chromium.chrome.browser.preferences.PreferencesLauncher;
 import org.chromium.chrome.browser.preferences.SearchUtils;
 import org.chromium.chrome.browser.preferences.TextMessagePreference;
+import org.chromium.chrome.browser.signin.IdentityServicesProvider;
 import org.chromium.ui.text.SpanApplier;
 
 import java.util.Locale;
@@ -477,6 +478,9 @@ public class SavePasswordsPreferences
         if (mSearchQuery != null && !mNoPasswords) {
             return; // Don't add the Manage Account link if there is a search going on.
         }
+        if (!IdentityServicesProvider.getSigninManager().isSigninSupported()) {
+            return;
+        }
         if (getPreferenceScreen().findPreference(PREF_KEY_MANAGE_ACCOUNT_LINK) != null) {
             return; // Don't add the Manage Account link if it's present.
         }
-- 
2.23.0

