From 57201adfadb127f5d782799f837d3b0778ee724d Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 24 Nov 2016 14:23:03 -0500
Subject: [PATCH 11/17] disable other data reduction proxy promotions

---
 .../browser/preferences/datareduction/DataReductionPromoUtils.java      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/preferences/datareduction/DataReductionPromoUtils.java b/chrome/android/java/src/org/chromium/chrome/browser/preferences/datareduction/DataReductionPromoUtils.java
index bd42c899b2f2..b147765261d3 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/preferences/datareduction/DataReductionPromoUtils.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/preferences/datareduction/DataReductionPromoUtils.java
@@ -68,7 +68,7 @@ public class DataReductionPromoUtils {
         }
         if (DataReductionProxySettings.getInstance().isDataReductionProxyManaged()) return false;
         if (DataReductionProxySettings.getInstance().isDataReductionProxyEnabled()) return false;
-        return true;
+        return false;
     }
 
     /**
-- 
2.11.1

