From 819d1cc77f959e3e950b2f43b8398439b3d6138d Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 8 Mar 2018 22:43:12 -0500
Subject: [PATCH 11/18] disable article suggestions feature by default

---
 components/ntp_snippets/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/ntp_snippets/features.cc b/components/ntp_snippets/features.cc
index a8bd7d167464..d950b1d5d30f 100644
--- a/components/ntp_snippets/features.cc
+++ b/components/ntp_snippets/features.cc
@@ -34,7 +34,7 @@ const base::Feature* const kAllFeatures[] = {
     &kRemoteSuggestionsBackendFeature};
 
 const base::Feature kArticleSuggestionsFeature{
-    "NTPArticleSuggestions", base::FEATURE_ENABLED_BY_DEFAULT};
+    "NTPArticleSuggestions", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kBookmarkSuggestionsFeature{
     "NTPBookmarkSuggestions", base::FEATURE_ENABLED_BY_DEFAULT};
-- 
2.21.0

