From 670bc6e797e13b79ffacc7a7995d574dbebb0f5a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 6 Mar 2018 00:27:41 -0500
Subject: [PATCH 11/19] disable showing popular sites by default

---
 components/ntp_tiles/constants.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/ntp_tiles/constants.cc b/components/ntp_tiles/constants.cc
index 9205d8093306..079ecb4009f0 100644
--- a/components/ntp_tiles/constants.cc
+++ b/components/ntp_tiles/constants.cc
@@ -13,7 +13,7 @@ namespace ntp_tiles {
 const char kPopularSitesFieldTrialName[] = "NTPPopularSites";
 
 const base::Feature kPopularSitesBakedInContentFeature{
-    "NTPPopularSitesBakedInContent", base::FEATURE_ENABLED_BY_DEFAULT};
+    "NTPPopularSitesBakedInContent", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kNtpMostLikelyFaviconsFromServerFeature{
     "NTPMostLikelyFaviconsFromServer", base::FEATURE_ENABLED_BY_DEFAULT};
@@ -22,7 +22,7 @@ const base::Feature kSiteExplorationUiFeature{
     "SiteExplorationUi", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kUsePopularSitesSuggestions{
-    "UsePopularSitesSuggestions", base::FEATURE_ENABLED_BY_DEFAULT};
+    "UsePopularSitesSuggestions", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kNtpCustomLinks{"NewTabPageCustomLinks",
                                     base::FEATURE_ENABLED_BY_DEFAULT};
-- 
2.20.1

