From bfa995d9bb8f8cfb7eb290c574769a18c7b46136 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 6 Mar 2018 00:27:41 -0500
Subject: [PATCH 11/19] disable showing popular sites by default

---
 components/ntp_tiles/constants.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/ntp_tiles/constants.cc b/components/ntp_tiles/constants.cc
index b40488bcd098..3cfb203d065a 100644
--- a/components/ntp_tiles/constants.cc
+++ b/components/ntp_tiles/constants.cc
@@ -12,7 +12,7 @@ namespace ntp_tiles {
 const char kPopularSitesFieldTrialName[] = "NTPPopularSites";
 
 const base::Feature kPopularSitesBakedInContentFeature{
-    "NTPPopularSitesBakedInContent", base::FEATURE_ENABLED_BY_DEFAULT};
+    "NTPPopularSitesBakedInContent", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kNtpMostLikelyFaviconsFromServerFeature{
     "NTPMostLikelyFaviconsFromServer", base::FEATURE_ENABLED_BY_DEFAULT};
@@ -21,6 +21,6 @@ const base::Feature kSiteExplorationUiFeature{
     "SiteExplorationUi", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kUsePopularSitesSuggestions{
-    "UsePopularSitesSuggestions", base::FEATURE_ENABLED_BY_DEFAULT};
+    "UsePopularSitesSuggestions", base::FEATURE_DISABLED_BY_DEFAULT};
 
 }  // namespace ntp_tiles
-- 
2.22.0

