From a78461c2960f36d846030f9473884538d1e49a08 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 8 Mar 2018 22:43:12 -0500
Subject: [PATCH 12/21] disable article suggestions feature by default

---
 components/ntp_snippets/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/ntp_snippets/features.cc b/components/ntp_snippets/features.cc
index d9bfeb9b3269..995c16e95c1f 100644
--- a/components/ntp_snippets/features.cc
+++ b/components/ntp_snippets/features.cc
@@ -41,7 +41,7 @@ const base::Feature* const kAllFeatures[] = {
     &kRemoteSuggestionsBackendFeature};
 
 const base::Feature kArticleSuggestionsFeature{
-    "NTPArticleSuggestions", base::FEATURE_ENABLED_BY_DEFAULT};
+    "NTPArticleSuggestions", base::FEATURE_DISABLED_BY_DEFAULT};
 
 const base::Feature kRemoteSuggestionsEmulateM58FetchingSchedule{
     "RemoteSuggestionsEmulateM58FetchingSchedule",
-- 
2.22.0

