From 632f6355794ca162ec344b15558a274e2122062f Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 26 Dec 2018 14:46:35 -0500
Subject: [PATCH 13/19] disable search engine notifications by default

---
 chrome/common/chrome_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
index 76af1c373362..f368a8811796 100644
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -291,7 +291,7 @@ const base::Feature kGeoLanguage{"GeoLanguage",
 
 #if defined(OS_ANDROID)
 const base::Feature kGrantNotificationsToDSE{"GrantNotificationsToDSE",
-                                             base::FEATURE_ENABLED_BY_DEFAULT};
+                                             base::FEATURE_DISABLED_BY_DEFAULT};
 #endif  // defined(OS_ANDROID)
 
 #if defined(OS_CHROMEOS)
-- 
2.20.1

