From 83032a8261b0bfc99814c9752b5c5815da2e7201 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sun, 13 Aug 2017 19:33:04 -0400
Subject: [PATCH 18/19] hide passwords.google.com link when not supported

---
 .../preferences/password/SavePasswordsPreferences.java        | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java b/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java
index c05e84931026..827ec79cf5b0 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/preferences/password/SavePasswordsPreferences.java
@@ -34,6 +34,7 @@ import org.chromium.chrome.browser.preferences.PrefServiceBridge;
 import org.chromium.chrome.browser.preferences.PreferencesLauncher;
 import org.chromium.chrome.browser.preferences.SearchUtils;
 import org.chromium.chrome.browser.preferences.TextMessagePreference;
+import org.chromium.chrome.browser.signin.SigninManager;
 import org.chromium.ui.text.SpanApplier;
 
 import java.util.Locale;
@@ -450,6 +451,9 @@ public class SavePasswordsPreferences
         if (mSearchQuery != null && !mNoPasswords) {
             return; // Don't add the Manage Account link if there is a search going on.
         }
+        if (!SigninManager.get().isSigninSupported()) {
+            return;
+        }
         if (getPreferenceScreen().findPreference(PREF_KEY_MANAGE_ACCOUNT_LINK) != null) {
             return; // Don't add the Manage Account link if it's present.
         }
-- 
2.20.1

