From 4ab7d7a33fd259a17e5492ba95338bb406c8a62a Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 16:56:37 -0400
Subject: [PATCH 27/28] stop ignoring download location prompt setting

---
 .../download/DownloadLocationDialogBridge.java     | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
index 64823b27d948..251558b0565f 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
@@ -109,20 +109,6 @@ public class DownloadLocationDialogBridge implements ModalDialogProperties.Contr
      * @param dirs An list of available download directories.
      */
     private void onDirectoryOptionsRetrieved(ArrayList<DirectoryOption> dirs) {
-        // If there is only one directory available, don't show the default dialog, and set the
-        // download directory to default. Dialog will still show for other types of dialogs, like
-        // name conflict or disk error.
-        if (dirs.size() == 1 && mDialogType == DownloadLocationDialogType.DEFAULT) {
-            final DirectoryOption dir = dirs.get(0);
-            if (dir.type == DirectoryOption.DownloadLocationDirectoryType.DEFAULT) {
-                assert(!TextUtils.isEmpty(dir.location));
-                PrefServiceBridge.getInstance().setDownloadAndSaveFileDefaultDirectory(
-                        dir.location);
-                nativeOnComplete(mNativeDownloadLocationDialogBridge, mSuggestedPath);
-            }
-            return;
-        }
-
         // Already showing the dialog.
         if (mDialogModel != null) return;
 
-- 
2.22.0

