From f007a0105798de422ee7d994bc94d4fe8c7c931d Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 17 Jun 2019 12:03:52 -0400
Subject: [PATCH 27/29] disable search provider logo

---
 .../template_url_service_android.cc           | 23 +------------------
 1 file changed, 1 insertion(+), 22 deletions(-)

diff --git a/chrome/browser/search_engines/template_url_service_android.cc b/chrome/browser/search_engines/template_url_service_android.cc
index df23c7b1ed5b..6662ccfa4df7 100644
--- a/chrome/browser/search_engines/template_url_service_android.cc
+++ b/chrome/browser/search_engines/template_url_service_android.cc
@@ -107,28 +107,7 @@ jboolean TemplateUrlServiceAndroid::IsDefaultSearchEngineGoogle(
 jboolean TemplateUrlServiceAndroid::DoesDefaultSearchEngineHaveLogo(
     JNIEnv* env,
     const JavaParamRef<jobject>& obj) {
-  // |kSearchProviderLogoURL| applies to all search engines (Google or
-  // third-party).
-  if (base::CommandLine::ForCurrentProcess()->HasSwitch(
-          search_provider_logos::switches::kSearchProviderLogoURL)) {
-    return true;
-  }
-
-  // Google always has a logo.
-  if (IsDefaultSearchEngineGoogle(env, obj))
-    return true;
-
-  // Third-party search engines can have a doodle specified via the command
-  // line, or a static logo or doodle from the TemplateURLService.
-  if (base::CommandLine::ForCurrentProcess()->HasSwitch(
-          search_provider_logos::switches::kThirdPartyDoodleURL)) {
-    return true;
-  }
-  const TemplateURL* default_search_provider =
-      template_url_service_->GetDefaultSearchProvider();
-  return default_search_provider &&
-         (default_search_provider->doodle_url().is_valid() ||
-          default_search_provider->logo_url().is_valid());
+  return false;
 }
 
 jboolean
-- 
2.22.0

