From e53caaf5a44e800b23d2837ef9a9d14624b6bb42 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 16:56:37 -0400
Subject: [PATCH 34/49] stop ignoring download location prompt setting

---
 .../download/DownloadLocationDialogBridge.java    | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
index 92513b019c76..44373ec4a963 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
@@ -109,21 +109,6 @@ public class DownloadLocationDialogBridge implements ModalDialogProperties.Contr
      * @param dirs An list of available download directories.
      */
     private void onDirectoryOptionsRetrieved(ArrayList<DirectoryOption> dirs) {
-        // If there is only one directory available, don't show the default dialog, and set the
-        // download directory to default. Dialog will still show for other types of dialogs, like
-        // name conflict or disk error.
-        if (dirs.size() == 1 && mDialogType == DownloadLocationDialogType.DEFAULT) {
-            final DirectoryOption dir = dirs.get(0);
-            if (dir.type == DirectoryOption.DownloadLocationDirectoryType.DEFAULT) {
-                assert(!TextUtils.isEmpty(dir.location));
-                setDownloadAndSaveFileDefaultDirectory(dir.location);
-                DownloadLocationDialogBridgeJni.get().onComplete(
-                        mNativeDownloadLocationDialogBridge, DownloadLocationDialogBridge.this,
-                        mSuggestedPath);
-            }
-            return;
-        }
-
         // Already showing the dialog.
         if (mDialogModel != null) return;
 
-- 
2.25.0

