From 100fdf28cb6a40abf16226bfb9f2b4a13d5f3c3c Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 16:56:37 -0400
Subject: [PATCH 34/49] stop ignoring download location prompt setting

---
 .../download/DownloadLocationDialogBridge.java   | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
index 6c1d7834fc55..191ad264f30c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationDialogBridge.java
@@ -110,22 +110,6 @@ public class DownloadLocationDialogBridge implements ModalDialogProperties.Contr
      * @param dirs An list of available download directories.
      */
     private void onDirectoryOptionsRetrieved(ArrayList<DirectoryOption> dirs) {
-        // If there is only one directory available, don't show the default dialog, and set the
-        // download directory to default. Dialog will still show for other types of dialogs, like
-        // name conflict or disk error.
-        if (dirs.size() == 1 && mDialogType == DownloadLocationDialogType.DEFAULT) {
-            final DirectoryOption dir = dirs.get(0);
-            if (dir.type == DirectoryOption.DownloadLocationDirectoryType.DEFAULT) {
-                assert(!TextUtils.isEmpty(dir.location));
-                PrefServiceBridge.getInstance().setDownloadAndSaveFileDefaultDirectory(
-                        dir.location);
-                DownloadLocationDialogBridgeJni.get().onComplete(
-                        mNativeDownloadLocationDialogBridge, DownloadLocationDialogBridge.this,
-                        mSuggestedPath);
-            }
-            return;
-        }
-
         // Already showing the dialog.
         if (mDialogModel != null) return;
 
-- 
2.24.0

