From 3344b42721579db45bffe9388fb7a0fe66198418 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 28 Jun 2019 17:48:49 -0400
Subject: [PATCH 35/49] show download prompt again by default

---
 .../chrome/browser/download/DownloadLocationCustomView.java   | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java
index 44beb6e3441d..4944192eefc9 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/download/DownloadLocationCustomView.java
@@ -56,10 +56,6 @@ public class DownloadLocationCustomView
     void initialize(@DownloadLocationDialogType int dialogType, File suggestedPath) {
         mDialogType = dialogType;
 
-        // Automatically check "don't show again" the first time the user is seeing the dialog.
-        boolean isInitial = PrefServiceBridge.getInstance().getPromptForDownloadAndroid()
-                == DownloadPromptStatus.SHOW_INITIAL;
-        mDontShowAgain.setChecked(isInitial);
         mDontShowAgain.setOnCheckedChangeListener(this);
 
         mFileName.setText(suggestedPath.getName());
-- 
2.24.1

