From dd46dfe4ebdf6d7d79d270fde65f32632a3dadb1 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 3 Aug 2019 00:28:49 -0400
Subject: [PATCH 44/48] disable contextual search by default

---
 .../browser/contextualsearch/ContextualSearchFieldTrial.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java b/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
index e62bebe8c820..8d0297c7edf2 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
@@ -287,7 +287,7 @@ public class ContextualSearchFieldTrial {
         // Allow disabling the feature remotely.
         if (getBooleanParam(DISABLED_PARAM)) return false;
 
-        return true;
+        return false;
     }
 
     /**
-- 
2.23.0

