From 5ab2fc9e622b63d63fb58d43dc86d91de6156f3e Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 3 Aug 2019 00:28:49 -0400
Subject: [PATCH 44/49] disable contextual search by default

---
 .../browser/contextualsearch/ContextualSearchFieldTrial.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java b/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
index 44061ca06a1e..49eb617d49f6 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
@@ -286,7 +286,7 @@ public class ContextualSearchFieldTrial {
         // Allow disabling the feature remotely.
         if (getBooleanParam(DISABLED_PARAM)) return false;
 
-        return true;
+        return false;
     }
 
     /**
-- 
2.24.1

