From c44ea5ef7f56b9afa4620e2f001bd2d12a191940 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Sat, 3 Aug 2019 00:28:49 -0400
Subject: [PATCH 45/47] disable contextual search by default

---
 .../browser/contextualsearch/ContextualSearchFieldTrial.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java b/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
index d1c3b35adece..2e758da20f31 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/contextualsearch/ContextualSearchFieldTrial.java
@@ -286,7 +286,7 @@ public class ContextualSearchFieldTrial {
         // Allow disabling the feature remotely.
         if (getBooleanParam(DISABLED_PARAM)) return false;
 
-        return true;
+        return false;
     }
 
     /**
-- 
2.21.0

